<!DOCTYPE html>
<html class="no-js" lang="en">
 <!-- Header -->
 <head>
  <title>
   Patent 3042857 Summary - Canadian Patents Database
  </title>
  <meta charset="utf-8"/>
  <meta content="IE=edge" http-equiv="X-UA-Compatible"/>
  <link href="https://www.ic.gc.ca/app/scr/opic-cipo/boew-wet/wet/4.0.28.1/themes-dist-gcweb/assets/favicon.ico" rel="shortcut icon"/>
  <meta content="en" name="dcterms.language" title="ISO639-2"/>
  <meta content="Canadian Patent Database / Base de données sur les brevets canadiens" name="dcterms.description"/>
  <meta content="Canadian Patent Database / Base de données sur les brevets canadiens" name="description"/>
  <meta content="Canadian Patent Database / Base de données sur les brevets canadiens" name="dcterms.description"/>
  <meta content="Government of Canada, Innovation, Science and Economic Development Canada, Office of the Deputy Minister, Canadian Intellectual Property Office" name="dcterms.creator"/>
  <meta content="Canadian Patent Database / Base de données sur les brevets canadiens" name="dcterms.title"/>
  <meta content="" name="dcterms.issued" title="W3CDTF"/>
  <meta content="2015-01-30" name="dcterms.modified" title="W3CDTF"/>
  <meta content="intellectual property" name="dcterms.subject" title="gccore"/>
  <meta content="width=device-width, initial-scale=1" name="viewport"/>
  <!-- Includes files. -->
  <link href="https://www.ic.gc.ca/app/scr/opic-cipo/boew-wet/wet/4.0.28.1/themes-dist-gcweb/assets/favicon.ico" rel="icon" type="image/x-icon"/>
  <link href="https://www.ic.gc.ca/app/scr/opic-cipo/boew-wet/wet/4.0.28.1/themes-dist-gcweb/css/wet-boew.min.css" rel="stylesheet"/>
  <noscript>
   <link href="https://www.ic.gc.ca/app/scr/opic-cipo/boew-wet/wet/4.0.28.1/themes-dist-gcweb/css/noscript.min.css" rel="stylesheet"/>
  </noscript>
  <!-- CustomCSSStart -->
  <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css" rel="stylesheet" type="text/css"/>
  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" type="text/css"/>
  <!-- CustomCSSEnd -->
  <link href="https://www.ic.gc.ca/app/scr/opic-cipo/boew-wet/wet/4.0.28.1/themes-dist-gcweb/css/theme.min.css" rel="stylesheet"/>
  <!-- CustomScriptsStart -->
  <!-- The following two scripts are for the data table. This was required because the WET 3.1 table didn't save the user's updates to the -->
  <!-- (sort, page number) when returning to this page using the Back Button.   -->
  <script src="https://www.ic.gc.ca/app/scr/opic-cipo/boew-wet/wet/4.0.28.1/themes-dist-gcweb/js/jquery/2.1.4/jquery.js">
  </script>
  <script src="https://www.ic.gc.ca/app/scr/opic-cipo/boew-wet/wet/4.0.28.1/themes-dist-gcweb/js/deps/jquery.dataTables.js">
  </script>
  <!-- CustomScriptsEnd -->
 </head>
 <!-- Body -->
 <body class="home" typeof="WebPage" vocab="http://schema.org">
  <!-- Hidden menu -->
  <ul id="wb-tphp">
   <li class="wb-slc">
    <a class="wb-sl" href="#wb-cont" title="Skip to main content">
     Skip to main content
    </a>
   </li>
  </ul>
  <!-- Header STARTS -->
  <header>
   <!-- Banner STARTS -->
   <div class="container" id="wb-bnr">
    <section class="text-right visible-md visible-lg" id="wb-lng">
     <h2 class="wb-inv">
      Language selection
     </h2>
     <div class="row">
      <div class="col-md-12">
       <ul class="list-inline mrgn-bttm-0">
        <li id="gcwu-gcnb-lang">
         <div>
          <a href="/opic-cipo/cpd/fra/brevet/3042857/sommaire.html?type=number_search&amp;tabs1Index=tabs1_1" lang="fr" title="Français - Version française de cette page">
           Français
          </a>
         </div>
        </li>
       </ul>
      </div>
     </div>
    </section>
    <div class="row">
     <div class="brand col-xs-8 col-sm-9 col-md-6">
      <a href="https://www.canada.ca/en.html" target="_blank" title="Government of Canada">
       <img src="https://www.ic.gc.ca/app/scr/opic-cipo/boew-wet/wet/4.0.28.1/themes-dist-gcweb/assets/sig-blk-en.svg"/>
       <span class="wb-inv">
        /
        <span lang="fr">
         Gouvernement du Canada
        </span>
       </span>
      </a>
     </div>
     <section class="col-xs-6 text-right visible-md visible-lg" id="wb-srch">
      <span class="wb-inv">
       Search
      </span>
      <form action="https://www.canada.ca/en/sr.html" class="form-inline" method="get" name="cse-search-box" role="search">
       <div class="form-group">
        <label class="wb-inv" for="wb-srch-q">
         https://www.canada.ca/en/sr.html
        </label>
        <input name="cdn" type="hidden" value="canada"/>
        <input name="st" type="hidden" value="s"/>
        <input name="num" type="hidden" value="10"/>
        <input name="langs" type="hidden" value="en"/>
        <input name="st1rt" type="hidden" value="1"/>
        <input name="s5bm3ts21rch" type="hidden" value="x"/>
        <input class="wb-srch-q form-control" id="wb-srch-q" list="wb-srch-q-ac" maxlength="150" name="q" placeholder="Search Canada.ca" size="27" type="search" value=""/>
        <input name="_charset_" type="hidden" value="UTF-8"/>
        <datalist id="wb-srch-q-ac">
        </datalist>
       </div>
       <div class="form-group submit">
        <button class="btn btn-primary btn-small" id="wb-srch-sub" name="wb-srch-sub" type="submit">
         <span class="glyphicon-search glyphicon">
         </span>
         <span class="wb-inv">
          Search
         </span>
        </button>
       </div>
      </form>
     </section>
     <!-- Search and Menu section -->
     <section class="wb-mb-links col-xs-4 col-sm-3 visible-sm visible-xs" id="wb-glb-mn">
      <h2>
       Menus
      </h2>
      <ul class="list-inline text-right chvrn">
       <li>
        <a aria-controls="mb-pnl" class="overlay-lnk" href="#mb-pnl" role="button" title="Menus">
         <span class="glyphicon glyphicon-th-list">
          <span class="wb-inv">
           Menus
          </span>
         </span>
        </a>
       </li>
      </ul>
      <div id="mb-pnl">
      </div>
     </section>
    </div>
   </div>
   <!-- Banner ENDS -->
   <!-- Menubar STARTS -->
   <nav class="wb-menu visible-md visible-lg" data-ajax-fetch="//cdn.canada.ca/gcweb-cdn-live/sitemenu/sitemenu-en.html" data-trgt="mb-pnl" id="wb-sm" typeof="SiteNavigationElement">
    <h2 class="wb-inv">
     Topics menu
    </h2>
    <div class="container nvbar">
     <div class="row">
      <ul class="list-inline menu">
       <li>
        <a href="https://www.esdc.gc.ca/en/jobs/index.page" rel="External">
         Jobs
        </a>
       </li>
       <li>
        <a href="https://www.cic.gc.ca/english/index.asp" rel="External">
         Immigration
        </a>
       </li>
       <li>
        <a href="https://travel.gc.ca" rel="External">
         Travel
        </a>
       </li>
       <li>
        <a href="https://www.canada.ca/en/services/business/index.html" rel="External">
         Business
        </a>
       </li>
       <li>
        <a href="https://www.canada.ca/en/services/benefits/index.html" rel="External">
         Benefits
        </a>
       </li>
       <li>
        <a href="https://healthycanadians.gc.ca/index-eng.php" rel="External">
         Health
        </a>
       </li>
       <li>
        <a href="https://www.canada.ca/en/services/taxes/index.html" rel="External">
         Taxes
        </a>
       </li>
       <li>
        <a href="https://www.canada.ca/en/services/index.html" rel="External">
         More services
        </a>
       </li>
      </ul>
     </div>
    </div>
   </nav>
   <!-- Menubar ENDS -->
   <!-- Bread Crumb STARTS -->
   <nav aria-labelledby="wb-bc" class="" id="wb-bc" property="breadcrumb">
    <div class="container">
     <div class="row">
      <div class="col-md-12">
       <ol class="breadcrumb">
        <li>
         <a href="https://www.canada.ca/en.html" rel="External">
          Home
         </a>
        </li>
        <li>
         <a href="https://www.canada.ca/en/services/business.html" rel="External">
          Business and industry
         </a>
        </li>
        <li>
         <a href="https://www.canada.ca/en/services/business/ip.html" rel="External">
          Intellectual property and copyright
         </a>
        </li>
        <li>
         <a href="https://www.canada.ca/en/services/business/ip/patents.html" rel="External">
          Patents
         </a>
        </li>
        <li>
         <abbr title="Canadian Patents Database">
          <a href="/opic-cipo/cpd/eng/introduction.html" title="The Introduction page of the Canadian Patents Database website">
           Canadian Patent Database
          </a>
         </abbr>
        </li>
        <li>
         <a href="/opic-cipo/cpd/eng/searchMenu.html" title="Search">
          Search
         </a>
        </li>
        <li>
         <a href="/opic-cipo/cpd/eng/search/number.html" title="Search the Canadian Patents Database using patent document number">
          Number Search
         </a>
        </li>
        <li>
         Patent Summary
        </li>
       </ol>
      </div>
     </div>
    </div>
   </nav>
   <!-- Bread Crumb ENDS -->
  </header>
  <!-- Header ENDS -->
  <div class="container">
   <!-- Main Content STARTS -->
   <div class="col-lg-9 col-lg-push-3">
    <div class="mrgn-tp-md">
     <!-- JSP pages STARTS -->
     <!-- Search term highlighting -->
     <link href="/opic-cipo/cpd/view/css/custom.css" media="screen, print" rel="stylesheet" type="text/css"/>
     <div class="section">
      <h1 id="wb-cont">
       Patent 3042857 Summary
      </h1>
      <details>
       <summary>
        <strong>
         Third-party information liability
        </strong>
       </summary>
       <div class="alert alert-info">
        <p>
         Some of the information on this Web page has been provided by external sources. The Government of Canada is not responsible for the accuracy, reliability or currency of the information supplied by external sources. Users wishing to rely upon this information should consult directly with the source of the information. Content provided by external sources is not subject to official languages, privacy and accessibility requirements.
        </p>
       </div>
      </details>
      <details>
       <summary>
        <strong>
         Claims and Abstract availability
        </strong>
       </summary>
       <div class="alert alert-info">
        <p>
         Any discrepancies in the text and image of the Claims and Abstract are due to differing posting times. Text of the Claims and Abstract are posted:
        </p>
        <ul>
         <li>
          At the time the application is open to public inspection;
         </li>
         <li>
          At the time of issue of the patent (grant).
         </li>
        </ul>
       </div>
      </details>
     </div>
     <div class="clearfix">
     </div>
     <script>
      //If Captcha entered is invalid, the following code will be executed
		function captchainvalid(){
			$('#catpcha').show();
			$('#catpchaInvalid').show();
			grecaptcha.reset();
			$('#mainSection').hide();
		}
     </script>
     <script type="text/javascript">
      var RecaptchaOptions = {
			    theme : 'white',
			    lang: 'en'
			 };
     </script>
     <!-- Patent Summary Table -->
     <table class="table table-bordered" id="patentSummaryTable" title="Table showing Patent Summary data">
      <tbody>
       <tr>
        <th class="col-lg-3 active" id="patentNum" scope="row">
         <a aria-label="(12) Patent ApplicationThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc12" id="aPatentNum12" onclick="saveFocusId();" title="The document may be an application or a patent">
          (12) Patent Application
         </a>
         :
        </th>
        <td headers="patentNum">
         <a aria-label="(11)This link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc11" id="aPatentNum11" onclick="saveFocusId();" title="Every application filed before october 1, 1989 has a separate application number and patent number">
          (11)
         </a>
         <strong>
          CA 3042857
         </strong>
        </td>
       </tr>
       <tr>
        <th class="active" id="EnglishTitle" scope="row">
         <a aria-label="(54) English TitleThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc54" id="aEnglishTitle" onclick="saveFocusId();" title="The english name the applicant have given to the application">
          (54) English Title
         </a>
         :
        </th>
        <td headers="EnglishTitle">
         METHODS FOR ALTERING AMINO ACID CONTENT IN PLANTS THROUGH FRAMESHIFT MUTATIONS
        </td>
       </tr>
       <tr>
        <th class="active" id="FrenchTitle" scope="row">
         <a aria-label="(54) French TitleThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc54" id="aFrenchTitle" onclick="saveFocusId();" title="The French name the applicants have given to the application">
          (54) French Title
         </a>
         :
        </th>
        <td headers="FrenchTitle" lang="fr">
         METHODES DE MODIFICATION DE LA TENEUR EN ACIDES AMINES DE PLANTES PAR DECALAGES DU CADRE DE LECTURE
        </td>
       </tr>
      </tbody>
     </table>
     <input id="tabs1Index" name="tabs1Index" type="hidden" value="tabs1_1"/>
     <div>
      <div class="accordion" id="tabs">
       <div>
        <!-- Bibliographic Data -->
        <details class="acc-group" id="tabs1_1" open="open">
         <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group", "persist": "session"}'>
          <span class="h6">
           Bibliographic Data
          </span>
         </summary>
         <div class="tgl-panel">
          <table class="table table-bordered" id="patentDetailsTable" title="Table showing Patent Details">
           <tbody>
            <tr>
             <th class="col-lg-5 active" id="intlClass">
              <a aria-label="(51) International Patent Classification (IPC)This link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc51" id="aIntClass" onclick="saveFocusId();" title="The classification system used by Canada and PCT member countries to categorize Canadian applications filed after August 15th, 1978">
               (51) International Patent Classification (IPC)
              </a>
              :
             </th>
             <td headers="intlClass">
              <ul class="list-unstyled mrgn-lft-0">
               <li>
                <span class="IPC-LEVEL-A IPC-VALUE-I">
                 C12N 15/82
                </span>
                (2006.01)
               </li>
              </ul>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="inventors">
              <a aria-label="(72) InventorsThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc72" id="aInventors" onclick="saveFocusId();" title="The names and country/territory of origin of each of the inventors of the application">
               (72) Inventors
              </a>
              :
             </th>
             <td headers="inventors">
              <ul class="list-unstyled mrgn-lft-0">
               <li>
                <strong>
                 BALTES, NICHOLAS
                </strong>
                (United States of America)
               </li>
               <li>
                <strong>
                 LUO, SONG
                </strong>
                (United States of America)
               </li>
              </ul>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="owners">
              <a aria-label="(73) OwnersThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc73" id="aOwners" onclick="saveFocusId();" title="The names and country/territory of origin of each of the current owners of the application or patent">
               (73) Owners
              </a>
              :
             </th>
             <td headers="owners">
              <ul class="list-unstyled mrgn-lft-0">
               <li>
                <strong>
                 CELLECTIS
                </strong>
                (France)
               </li>
              </ul>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="applicants">
              <a aria-label="(71) ApplicantsThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc71" id="aApplicants" onclick="saveFocusId();" title="Type an applicant name in the box to find patent applications associated with that applicant">
               (71) Applicants
              </a>
              :
             </th>
             <td headers="applicants">
              <ul class="list-unstyled mrgn-lft-0">
               <li>
                <strong>
                 CELLECTIS
                </strong>
                (France)
               </li>
              </ul>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="agent">
              <a aria-label="(74) AgentThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc74" id="aAgent" onclick="saveFocusId();" title="The name of the agent or agent company representing the owner in the Canadian Patent Office ">
               (74) Agent
              </a>
              :
             </th>
             <td headers="agent">
              <strong>
               AIRD &amp; MCBURNEY LP
              </strong>
             </td>
            </tr>
            <!-- Associate Agent -->
            <tr>
             <th class="col-lg-5 active" id="associateAgent">
              <a aria-label="Associate agentThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc74" id="aAssociateAgent" onclick="saveFocusId();" title="The name of the agent or agent company representing the owner in the Canadian Patent Office ">
               (74) Associate agent
              </a>
              :
             </th>
             <td headers="associateAgent">
              <strong>
              </strong>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="issued">
              <a aria-label="(45) IssuedThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc45" id="aIssued" onclick="saveFocusId();" title="Date an application was granted patent status in Canada">
               (45) Issued
              </a>
              :
             </th>
             <td headers="issued">
              <strong>
              </strong>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="filingDate">
              <a aria-label="(86) PCT Filing DateThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#pct" id="aFilingDate" onclick="saveFocusId();" title="The date the application was filed with WIPO">
               (86) PCT Filing Date
              </a>
              :
             </th>
             <td headers="filingDate">
              <strong>
               2017-11-16
              </strong>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="pubDate">
              <a aria-label="(87) Open to Public InspectionThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#pct" id="aPublicationDate" onclick="saveFocusId();" title="Date the application was made available for public inspection (viewing by the public)">
               (87) Open to Public Inspection
              </a>
              :
             </th>
             <td headers="pubDate">
              <strong>
               2018-05-24
              </strong>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="lic">
              <a aria-label="Availability of licenceThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#lic" id="aLicense" onclick="saveFocusId();" title="Patents where the owner voluntarily indicated that (s)he is willing to sell or license the rights to the patent.">
               Availability of licence
              </a>
              :
             </th>
             <td headers="lic">
              <strong>
               N/A
              </strong>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="lang">
              <a aria-label="(25) Language of filingThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc25" id="aLanguage" onclick="saveFocusId();" title="Indicates whether an application was filed in english or french.  This applies to applications filed in Canada after August 15th, 1978">
               (25) Language of filing
              </a>
              :
             </th>
             <td headers="lang">
              <strong>
               English
              </strong>
             </td>
            </tr>
           </tbody>
          </table>
          <br/>
          <div class="clearfix">
          </div>
          <!-- PCT Table -->
          <table class="table table-bordered col-lg-12" id="pctTable" title="Table showing Patent Cooperation Treaty details">
           <tbody>
            <tr>
             <th class="col-lg-5 active" id="pct">
              <a aria-label="Patent Cooperation Treaty (PCT)This link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#pct" id="aPct" onclick="saveFocusId();" title="An application which was filed in a PCT member country and Canada (andperhaps other countries)">
               Patent Cooperation Treaty (PCT)
              </a>
              :
             </th>
             <td headers="pct">
              <strong>
               Yes
              </strong>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="pctNum">
              <a aria-label="(86) PCT Filing NumberThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#pct" id="aPctNum" onclick="saveFocusId();" title="The number assigned to the application by WIPO">
               (86) PCT Filing Number
              </a>
              :
             </th>
             <td headers="pctNum">
              <strong>
               PCT/IB2017/057190
              </strong>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="pubNum">
              <a aria-label="(87) International Publication NumberThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#pct" id="aPubNum" onclick="saveFocusId();" title="The number assigned to the application by WIPO and the date the application was filed with WIPO">
               (87) International Publication Number
              </a>
              :
             </th>
             <td headers="pubNum">
              <strong>
               WO2018/092072
              </strong>
             </td>
            </tr>
            <tr>
             <th class="col-lg-5 active" id="national">
              <a aria-label="(85) National EntryThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#pct" id="aNational" onclick="saveFocusId();" title="The date the PCT application was given a Canadian filing date (entered the Canadian phase)">
               (85) National Entry
              </a>
              :
             </th>
             <td headers="national">
              <strong>
               2019-05-03
              </strong>
             </td>
            </tr>
           </tbody>
          </table>
          <br/>
          <div class="clearfix">
          </div>
          <!-- Application Priority Table -->
          <table class="table table-bordered col-lg-12" id="ApplicationPriorityTable" title="Table showing Application Priority Data">
           <tbody>
            <tr>
             <th class="col-lg-12 active text-center" id="appPriorityData" scope="col">
              <a aria-label="(30) Application Priority DataThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_bibliographic_data.html#Anc30" id="aAppPriorityData" onclick="saveFocusId();" title="Application number, country/territory of filing, and filing date of an application filed previously in another country/territory">
               (30) Application Priority Data
              </a>
              :
             </th>
            </tr>
            <tr>
             <td headers="appPriorityData">
              <table class="table table-bordered col-lg-12" id="appPriorityTable" title="Application Priority Data">
               <thead>
                <tr>
                 <th class="active" id="appNumber" scope="col">
                  Application No.
                 </th>
                 <th class="active" id="country" scope="col">
                  Country/Territory
                 </th>
                 <th class="active" id="date" scope="col">
                  Date
                 </th>
                </tr>
               </thead>
               <tbody>
                <tr>
                 <td headers="appNumber">
                  62/422,854
                 </td>
                 <td headers="country">
                  United States of America
                 </td>
                 <td headers="date">
                  2016-11-16
                 </td>
                </tr>
                <tr>
                 <td headers="appNumber">
                  62/485,001
                 </td>
                 <td headers="country">
                  United States of America
                 </td>
                 <td headers="date">
                  2017-04-13
                 </td>
                </tr>
               </tbody>
              </table>
             </td>
            </tr>
           </tbody>
          </table>
          <br/>
          <div class="clearfix">
          </div>
         </div>
        </details>
        <!-- Abstract -->
        <details class="acc-group" id="tabs1_2">
         <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group", "persist": "session"}'>
          <span class="h6">
           Abstracts
          </span>
         </summary>
         <div class="tgl-panel">
          <a id="View_Abstract">
          </a>
          <h2>
           English Abstract
          </h2>
          <p>
           Materials and methods are provided for making plants with altered levels of amino acids, particularly by making controlled frameshift mutations in genes that are highly expressed in plant leaves or plant seeds.
          </p>
          <br/>
          <h2>
           French Abstract
          </h2>
          <div lang="fr">
           <p>
            L'invention concerne des matériaux et des méthodes pour fabriquer des plantes à taux modifiés d'acides aminés, notamment par décalages du cadre de lecture dans des gènes qui sont fortement exprimés dans des feuilles de plantes ou des graines de plantes.
           </p>
          </div>
         </div>
        </details>
        <!-- Claims -->
        <details class="acc-group" id="tabs1_3">
         <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group", "persist": "session"}'>
          <span class="h6">
           Claims
          </span>
         </summary>
         <div class="tgl-panel">
          <div class="mrgn-lft-md">
           <div class="alert alert-info">
            <p>
             Note: Claims are shown in the official language in which they were submitted.
            </p>
           </div>
           <div style="height: 400px; overflow-y: scroll">
            <br/>
            33
            <br/>
            WHAT IS CLAIMED IS:
            <br/>
            1.    A method for altering the amino acid content of a polypeptide,
            <br/>
            comprising
            <br/>
            evaluating two or more reading frames within a nucleic acid encoding the
            <br/>
            polypeptide,
            <br/>
            identifying a reading frame that encodes an amino acid sequence having a
            <br/>
            desired amino acid content, and
            <br/>
            introducing a frameshift mutation into the nucleic acid such that when the
            <br/>
            nucleic acid sequence is expressed in a cell, the polypeptide having the
            <br/>
            desired amino
            <br/>
            acid content is expressed.
            <br/>
            2.    The method of claim 1, wherein the frameshift mutation is of the size -
            <br/>
            3(N) ¨
            <br/>
            2.
            <br/>
            3.    The method of claim 1, wherein the frameshift mutation is of the size
            <br/>
            +3(N) +
            <br/>
            1.
            <br/>
            4.    The method of claim 1, comprising contacting the nucleic acid with a
            <br/>
            rare-
            <br/>
            cutting endonuclease to introduce the frameshift mutation.
            <br/>
            5.    The method of claim 4, wherein the rare-cutting endonuclease is a
            <br/>
            transcription activator-like effector endonuclease (TALE nuclease), a
            <br/>
            meganuclease, a
            <br/>
            zinc finger nuclease (ZFN), or a clustered regularly interspaced short
            <br/>
            palindromic
            <br/>
            repeats (CRISPR)/CR1SPR-associated (Cas) nuclease reagent.
            <br/>
            6.    The method of claim 1, wherein the polypeptide encoded by the nucleic
            <br/>
            acid
            <br/>
            containing the frameshift mutation has increased sulfur-containing amino acid
            <br/>
            content
            <br/>
            as compared to a corresponding wild type polypeptide.
            <br/>
            7.    The method of claim 6, wherein the nucleic acid encodes a soybean
            <br/>
            globulin
            <br/>
            polypeptide, and wherein the frameshift mutation is within the sequence set
            <br/>
            forth in
            <br/>
            SEQ ID NO:94, or a sequence having at least 90% identity to SEQ ID NO:94.
            <br/>
            8.    The method of claim 6, wherein the polypeptide encoded by the nucleic
            <br/>
            acid
            <br/>
            containing the frameshift mutation is a soybean globulin polypeptide that
            <br/>
            comprises
            <br/>
            the amino acid sequence set forth in SEQ ID NO:95.
            <br/>
            <br/>
            34
            <br/>
            9.    The method of claim 1, wherein the polypeptide encoded by the nucleic
            <br/>
            acid
            <br/>
            containing the frameshift mutation has increased threonine content as compared
            <br/>
            to a
            <br/>
            corresponding wild type polypeptide.
            <br/>
            10.    The method of claim 9, wherein the nucleic acid encodes a wheat alpha
            <br/>
            gliadin
            <br/>
            polypeptide, and wherein the frameshift mutation is within the sequence set
            <br/>
            forth in
            <br/>
            SEQ ID NO:96, or a sequence haying at least 90% identity to SEQ ID NO:96.
            <br/>
            11.    The method of claim 9, wherein the polypeptide encoded by the nucleic
            <br/>
            acid
            <br/>
            containing the frameshift mutation is a wheat alpha gliadin polypeptide that
            <br/>
            comprises
            <br/>
            the amino acid sequence set forth in SEQ ID NO:97, or an amino acid sequence
            <br/>
            haying at least 90% sequence identity to SEQ ID NO:97.
            <br/>
            12.    The method of claim 9, wherein the nucleic acid encodes a wheat high
            <br/>
            molecular weight glutenin polypeptide, and wherein the frameshift mutation is
            <br/>
            within
            <br/>
            the sequence set forth in SEQ ID NO:70, or a sequence haying at least 90%
            <br/>
            identity to
            <br/>
            SEQ ID NO:70.
            <br/>
            13.    The method of claim 12, wherein the frameshift mutation encompasses or
            <br/>
            is 3'
            <br/>
            to the nucleotide at position 171 of SEQ ID NO:70.
            <br/>
            14.    The method of claim 9, wherein the polypeptide encoded by the nucleic
            <br/>
            acid
            <br/>
            containing the frameshift mutation is a wheat high molecular weight glutenin
            <br/>
            polypeptide that comprises the amino acid sequence set forth in SEQ ID NO:98,
            <br/>
            or an
            <br/>
            amino acid sequence haying at least 90% identity to SEQ ID NO:98.
            <br/>
            15.    The method of claim 1, wherein the polypeptide encoded by the nucleic
            <br/>
            acid
            <br/>
            containing the frameshift mutation has increased lysine content as compared to
            <br/>
            a
            <br/>
            corresponding wild type polypeptide.
            <br/>
            16.    The method of claim 15, wherein the nucleic acid encodes a wheat high
            <br/>
            molecular weight glutenin polypeptide, and wherein the frameshift mutation is
            <br/>
            within
            <br/>
            the sequence set forth in SEQ ID NO:70, or a sequence haying at least 90%
            <br/>
            identity to
            <br/>
            SEQ ID NO:70.
            <br/>
            <br/>
            35
            <br/>
            17.    The method of claim 16, wherein the frameshift mutation encompasses or
            <br/>
            is 3'
            <br/>
            to the nucleotide at position 348 of SEQ ID NO:70.
            <br/>
            18.    The method of claim 15, wherein the polypeptide encoded by the nucleic
            <br/>
            acid
            <br/>
            containing the frameshift mutation is a wheat high molecular weight glutenin
            <br/>
            polypeptide that comprises the amino acid sequence set forth in SEQ ID NO:99,
            <br/>
            or an
            <br/>
            amino acid sequence having at least 90% identity to SEQ ID NO:99.
            <br/>
            19.    The method of claim 1, further comprising introducing a second
            <br/>
            frameshift
            <br/>
            mutation into the nucleic acid encoding the polypeptide, wherein the
            <br/>
            frameshift
            <br/>
            mutations in combination result in a deletion or insertion of nucleotides, and
            <br/>
            wherein
            <br/>
            the size of the deletion or insertion is a multiple of 3.
            <br/>
            20.    A method for generating a plant, plant part, or plant cell with altered
            <br/>
            levels of
            <br/>
            amino acids, the method comprising:
            <br/>
            (a) contacting a plant, plant part, or plant cell with a rare-cutting
            <br/>
            endonuclease
            <br/>
            targeted to a sequence within an exon of a gene endogenous to the plant, plant
            <br/>
            part, or
            <br/>
            plant cell, such that the rare-cutting endonuclease generates a double strand
            <br/>
            break at
            <br/>
            or near the sequence to which it is targeted; and
            <br/>
            (b) selecting a plant, plant part, or plant cell that contains a frameshift
            <br/>
            mutation within the exon, wherein the plant, plant part, or plant cell has
            <br/>
            altered amino
            <br/>
            acid levels as compared to a control plant, plant part, or plant cell in which
            <br/>
            the
            <br/>
            frameshift mutation was not introduced.
            <br/>
            21.    The method of claim 20, further comprising growing a plant part or
            <br/>
            plant cell
            <br/>
            selected in step (b) into a plant.
            <br/>
            22.    The method of claim 20, wherein the plant cell that is contacted in
            <br/>
            step (a) is a
            <br/>
            protoplast.
            <br/>
            23.    The method of claim 22, wherein the method comprises transforming the
            <br/>
            protoplast with a nucleic acid encoding the rare-cutting endonuclease.
            <br/>
            24.    The method of claim 23, wherein the nucleic acid is an mRNA.
            <br/>
            25.    The method of claim 23 wherein the nucleic acid is contained within a
            <br/>
            vector.
            <br/>
            <br/>
            36
            <br/>
            26.    The method of claim 19, wherein the plant part that is contacted in
            <br/>
            step (a) is
            <br/>
            an immature embryo or embryogenic callus.
            <br/>
            27.    The method of claim 26, wherein the method comprises transforming the
            <br/>
            embryo or embryogenic callus with a nucleic acid encoding the rare-cutting
            <br/>
            endonuclease.
            <br/>
            28.    The method of claim 26, wherein the transforming comprises
            <br/>
            Agrobacterium-
            <br/>
            mediated transformation or biolistics.
            <br/>
            29.    The method of claim 19, wherein the rare-cutting endonuclease is a TALE
            <br/>
            <br/>
            nuclease, meganuclease, ZFN, or CRISPR/Cas nuclease reagent.
            <br/>
            30.    The method of claim 22 or claim 26, further comprising culturing the
            <br/>
            protoplasts, immature embryos, or embryogenic calli to generate plant lines.
            <br/>
            31.    The method of claim 19, wherein the frameshift mutation is within the
            <br/>
            last
            <br/>
            exon of the gene.
            <br/>
            32.    The method of claim 19, wherein the frameshift is in the coding
            <br/>
            sequence of
            <br/>
            the gene.
            <br/>
            33.    The method of claim 19, wherein the frameshift is introduced by
            <br/>
            homologous
            <br/>
            recombination with a user-supplied donor molecule.
            <br/>
            34.    The method of claim 19, wherein the frameshift mutation is within a
            <br/>
            gene that
            <br/>
            encodes a seed storage protein.
            <br/>
            35.    The method of claim 34, wherein the seed storage protein is selected
            <br/>
            from the
            <br/>
            group consisting of gliadin, hordein, secalin, zein, kafirin, avenin,
            <br/>
            glycinin, and
            <br/>
            conglycinin.
            <br/>
            36.    The method of claim 34, wherein the seed storage protein encoded by the
            <br/>
            gene
            <br/>
            containing the frameshift mutation comprises the amino acid sequence set forth
            <br/>
            in
            <br/>
            SEQ ID NO:95, SEQ ID NO:98, or SEQ ID NO:99, or an amino acid sequence
            <br/>
            having at least 90% identity to SEQ ID NO:95, SEQ ID NO:98, or SEQ ID NO:99.
            <br/>
            <br/>
            37
            <br/>
            37.    The method of claim 19, wherein the frameshift mutation is within a
            <br/>
            gene that
            <br/>
            encodes a protein expressed in leaf tissue.
            <br/>
            38.    The method of claim 37, wherein the protein expressed in leaf tissue is
            <br/>
            <br/>
            ribulose-1,5-bisphosphate (RuBP) carboxylase/oxygenase (rubisco),
            <br/>
            translational
            <br/>
            elongation factor EF-1 alpha (EF1a), or ubiquitin.
            <br/>
            39.    A method for generating a plant, plant part, or plant cell with altered
            <br/>
            levels of
            <br/>
            amino acids, the method comprising:
            <br/>
            (a) contacting a plant, plant part, or plant cell with a first rare-cutting
            <br/>
            endonuclease targeted to a sequence within a gene endogenous to the plant,
            <br/>
            plant part,
            <br/>
            or plant cell, such that the first rare-cutting endonuclease generates a
            <br/>
            double strand
            <br/>
            break at or near the sequence to which it is targeted;
            <br/>
            (b) selecting a plant, plant part, or plant cell that contains a first
            <br/>
            frameshift
            <br/>
            mutation within the gene;
            <br/>
            (c) contacting a plant, plant part or plant cell with a second rare-cutting
            <br/>
            endonuclease targeted to a sequence within the same gene as that to which the
            <br/>
            first
            <br/>
            rare-cutting endonuclease was targeted, such that the second rare-cutting
            <br/>
            endonuclease generates a double strand break at or near the sequence to which
            <br/>
            it is
            <br/>
            targeted; and
            <br/>
            (d) selecting a plant, plant part, or plant cell that contains a second
            <br/>
            mutation
            <br/>
            within the endogenous gene.
            <br/>
            40.    The method of claim 39, wherein the plant cell that is contacted in
            <br/>
            step (a) or
            <br/>
            step (c) is a protoplast.
            <br/>
            41.    The method of claim 40, comprising transforming the protoplast with a
            <br/>
            nucleic
            <br/>
            acid encoding the first or second rare-cutting endonuclease.
            <br/>
            42.    The method of claim 41, wherein the nucleic acid is an mRNA.
            <br/>
            43.    The method of claim 41, wherein the nucleic acid is contained within a
            <br/>
            vector.
            <br/>
            44.    The method of claim 39, wherein the plant part that is contacted in
            <br/>
            step (a) or
            <br/>
            step (c) is an immature embryo or embryogenic callus.
            <br/>
            <br/>
            38
            <br/>
            45.    The method of claim 44, comprising transforming the embryo or
            <br/>
            embryogenic
            <br/>
            callus with a nucleic acid encoding the first or second rare-cutting
            <br/>
            endonuclease.
            <br/>
            46.    The method of claim 45, wherein the transforming comprises
            <br/>
            Agrobacterium-
            <br/>
            mediated transformation or transformation by biolistics.
            <br/>
            47.    The method of claim 39, wherein the first or second rare-cutting
            <br/>
            endonuclease
            <br/>
            is a TALE nuclease, meganuclease, ZFN, or CR1SPR/Cas reagent.
            <br/>
            48.    The method of claim 40 or claim 44, further comprising culturing the
            <br/>
            protoplast, immature embryo, or embryogenic callus to generate a plant line.
            <br/>
            49.    The method of claim 39, wherein the first frameshift mutation is
            <br/>
            introduced
            <br/>
            chronologically before the second mutation, and wherein the second mutation is
            <br/>
            <br/>
            introduced into a plant, plant part, or plant cell selected in step (b).
            <br/>
            50.    The method of claim 39, wherein the second mutation is introduced
            <br/>
            chronologically before the first frameshift mutation, and wherein the first
            <br/>
            frameshift
            <br/>
            mutation is introduced into a plant, plant part, or plant cell selected in
            <br/>
            step (d).
            <br/>
            51.    The method of claim 39, wherein the first frameshift mutation is within
            <br/>
            an
            <br/>
            exon of the gene.
            <br/>
            52.    The method of claim 39, wherein the second mutation is downstream of
            <br/>
            the
            <br/>
            first frameshift mutation.
            <br/>
            53.    The method of claim 39, wherein the second mutation is a frameshift
            <br/>
            mutation
            <br/>
            that re-introduces the normal reading frame found in the wild type gene.
            <br/>
            54.    The method of claim 39, wherein the second mutation inactivates
            <br/>
            splicing of
            <br/>
            introns downstream from the first frameshift mutation.
            <br/>
            55.    The method of claim 39, wherein the first frameshift mutation or the
            <br/>
            second
            <br/>
            mutation is introduced by homologous recombination using a user-generated
            <br/>
            donor
            <br/>
            molecule.
            <br/>
            <br/>
            39
            <br/>
            56.    The method of claim 39, wherein the first frameshift mutation and the
            <br/>
            second
            <br/>
            mutation are introduced simultaneously by homologous recombination using a
            <br/>
            user-
            <br/>
            generated donor molecule, or by simultaneously delivering two or more rare-
            <br/>
            cutting
            <br/>
            endonucleases.
            <br/>
            57.    The method of claim 39, wherein the gene encodes a seed storage
            <br/>
            protein.
            <br/>
            58.    The method of claim 57, wherein the seed storage protein is selected
            <br/>
            from the
            <br/>
            group consisting of gliadin, hordein, secalin, zein, kafirin, avenin,
            <br/>
            glycinin, and
            <br/>
            conglycinin.
            <br/>
            59.    The method of claim 57, wherein the seed storage protein encoded by the
            <br/>
            gene
            <br/>
            containing the frameshift mutation comprises the amino acid sequence set forth
            <br/>
            in
            <br/>
            SEQ ID NO:95, SEQ ID NO:98, or SEQ ID NO:99, or an amino acid sequence
            <br/>
            having at least 90% identity to SEQ ID NO:95, SEQ ID NO:98, or SEQ ID NO:99.
            <br/>
            60.    The method of claim 39, wherein the gene encodes a protein that is
            <br/>
            expressed
            <br/>
            in leaf tissue.
            <br/>
            61.    The method of claim 60, wherein the protein expressed in leaf tissue is
            <br/>
            <br/>
            rubisco, EF1a, or ubiquitin.
            <br/>
            62.    A plant, plant part, or plant cell with altered levels of amino acids,
            <br/>
            wherein the
            <br/>
            plant contains a frameshift mutation in an exon of a selected gene.
            <br/>
            63.    The plant, plant part, or plant cell of claim 62, wherein the altered
            <br/>
            levels of
            <br/>
            amino acids comprise at least a 0.1% increase or decrease in the content of
            <br/>
            one or
            <br/>
            more amino acids.
            <br/>
            64.    The plant, plant part, or plant cell of claim 62, wherein the plant,
            <br/>
            plant part, or
            <br/>
            plant cell contains a second frameshift mutation within the selected gene.
            <br/>
            65.    The plant, plant part, or plant cell of claim 62, wherein the plant,
            <br/>
            plant part, or
            <br/>
            plant cell comprises a second mutation within an exon or intron of the
            <br/>
            selected gene.
            <br/>
            <br/>
            40
            <br/>
            66.    The plant, plant part, or plant cell of claim 65, wherein the second
            <br/>
            mutation is
            <br/>
            a deletion, insertion, substitution, or inversion of nucleotides that are
            <br/>
            required for
            <br/>
            intron splicing.
            <br/>
            67.    The plant, plant part, or plant cell of claim 62, wherein the plant,
            <br/>
            plant part, or
            <br/>
            plant cell is a wheat, cassava, alfalfa, oat, corn, rice, sorghum, potato,
            <br/>
            tomato,
            <br/>
            soybean, or canola plant, plant part, or plant cell.
            <br/>
            <br/>
           </div>
          </div>
         </div>
        </details>
        <!-- Description -->
        <details class="acc-group" id="tabs1_3b">
         <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group", "persist": "session"}'>
          <span class="h6">
           Description
          </span>
         </summary>
         <div class="tgl-panel">
          <div class="mrgn-lft-md">
           <div class="alert alert-info">
            <p>
             Note: Descriptions are shown in the official language in which they were submitted.
            </p>
           </div>
           <div style="height: 400px; overflow-y: scroll">
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            1
            <br/>
            METHODS FOR ALTERING AMINO ACID CONTENT IN PLANTS
            <br/>
            THROUGH FRAMESHIFT MUTATIONS
            <br/>
            CROSS-REFERENCE TO RELATED APPLICATIONS
            <br/>
            This application claims benefit of priority from U.S. Provisional Application
            <br/>
            Serial No. 62/485,001, filed on April 13, 2017, and U.S. Provisional
            <br/>
            Application Serial
            <br/>
            No. 62/422,854, filed on November 16, 2016.
            <br/>
            TECHNICAL FIELD
            <br/>
            This document provides materials and methods for generating plants with
            <br/>
            altered
            <br/>
            levels of amino acids.
            <br/>
            BACKGROUND
            <br/>
            Humans, as well as farm animals, are unable to synthesize several amino acids
            <br/>
            that are required for survival, including histidine, isoleucine, leucine,
            <br/>
            methionine,
            <br/>
            phenylalanine, threonine, tryptophan, valine, and lysine. As a result, the
            <br/>
            diet of humans
            <br/>
            and farm animals must contain sufficient levels of these essential amino
            <br/>
            acids. In
            <br/>
            developed countries, optimal levels of essential amino acids are generally
            <br/>
            achieved
            <br/>
            through diets consisting of meat, eggs, milk, cereals, and legumes. However,
            <br/>
            in
            <br/>
            developing countries, diets are frequently restricted to major crop plants,
            <br/>
            which can result
            <br/>
            in a deficiency of particular amino acids. Suboptimal levels of essential
            <br/>
            amino acids can
            <br/>
            lead to protein-energy malnutrition (PEM), which is characterized by increased
            <br/>
            <br/>
            susceptibility to disease, decreased levels of blood proteins, and impaired
            <br/>
            mental and
            <br/>
            physical development in children. It is estimated by the World Health
            <br/>
            Organization that
            <br/>
            30% of the population in developing countries suffer from PEM (Onis et al.,
            <br/>
            Bull World
            <br/>
            Health Organ, 71: 703-712, 1993).
            <br/>
            SUMMARY
            <br/>
            This document provides materials and methods for generating plants with
            <br/>
            altered
            <br/>
            (e.g., increased) levels of particular amino acids. For example, this document
            <br/>
            relates to
            <br/>
            the use of genome engineering tools (e.g., sequence-specific nucleases and
            <br/>
            donor
            <br/>
            molecules) to generate controlled frameshift mutations that lead to altered
            <br/>
            amino acid
            <br/>
            content in plants that are modified using the tools. The methods described
            <br/>
            herein can be
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            2
            <br/>
            useful to, for example, fortify major crop plants with increased levels of
            <br/>
            essential amino
            <br/>
            acids, thus providing the potential to improve human health. Further, plants
            <br/>
            containing
            <br/>
            genome modifications introduced by sequence-specific nucleases are not
            <br/>
            regulated in
            <br/>
            certain jurisdictions; therefore, this is considered a non-transgenic approach
            <br/>
            to improving
            <br/>
            the amino acid content in crop plants.
            <br/>
            This disclosure is based at least in part on the discovery that plants with
            <br/>
            altered
            <br/>
            amino acid content can be obtained using sequence-specific nucleases to
            <br/>
            generate
            <br/>
            controlled frameshift mutations. Specifically, it has been determined that (i)
            <br/>
            small
            <br/>
            deletions or insertions can result in frameshift mutations, (ii) sequence-
            <br/>
            specific nucleases
            <br/>
            with or without a donor molecule can generate targeted frameshift mutations,
            <br/>
            and (iii)
            <br/>
            codons within alternative reading frames can encode valuable amino acids. In
            <br/>
            some
            <br/>
            embodiments, the methods provided herein can involve the design and delivery
            <br/>
            of
            <br/>
            sequence-specific nucleases targeting coding sequence within a gene of
            <br/>
            interest.
            <br/>
            Erroneous repair of the resulting double-strand break by non-homologous end
            <br/>
            joining
            <br/>
            (NHEJ) can result in a frameshift mutation, which can subsequently lead to a
            <br/>
            premature
            <br/>
            stop codon and a truncated protein. As described herein, frameshift mutations
            <br/>
            also can be
            <br/>
            used to modulate the amino acid composition of proteins, and ultimately, the
            <br/>
            amino acid
            <br/>
            content in modified plants. Controlled frameshift mutations within genes that
            <br/>
            are highly
            <br/>
            expressed (e.g., seed storage protein genes, including gliadin, hordein,
            <br/>
            secalin, zein,
            <br/>
            kafirin, avenin, glycinin, and conglycinin), can result in the production of
            <br/>
            proteins with
            <br/>
            significantly higher levels of one or more amino acids of interest.
            <br/>
            In addition, this document is based at least in part on the development of
            <br/>
            crop
            <br/>
            varieties with mutations in seed storage proteins, or other highly expressed
            <br/>
            genes, where
            <br/>
            the mutations are created using sequence-specific nucleases. The methods
            <br/>
            provided
            <br/>
            herein for modulating amino acid content can be achieved without insertion of
            <br/>
            a
            <br/>
            transgene. In addition, the materials and methods provided herein can address
            <br/>
            challenges
            <br/>
            associated with commercializing transgenic plants, including strict regulation
            <br/>
            in certain
            <br/>
            jurisdictions, and high costs to obtain regulatory approval. The methods
            <br/>
            described herein
            <br/>
            can accelerate the production of new crop varieties with modified levels of
            <br/>
            amino acids,
            <br/>
            and can be more cost effective than transgenic or traditional breeding
            <br/>
            approaches.
            <br/>
            In one aspect, this document features a method for altering the amino acid
            <br/>
            content
            <br/>
            of a polypeptide. The method can include evaluating two or more reading frames
            <br/>
            within a
            <br/>
            nucleic acid encoding the polypeptide, identifying a reading frame that
            <br/>
            encodes an amino
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            3
            <br/>
            acid sequence having a desired amino acid content, and introducing a
            <br/>
            frameshift mutation
            <br/>
            into the nucleic acid such that when the nucleic acid sequence is expressed in
            <br/>
            a cell, the
            <br/>
            polypeptide having the desired amino acid content is expressed. The frameshift
            <br/>
            mutation
            <br/>
            can be of the size -3(N) ¨ 2, or the size +3(N) + 1. The method can include
            <br/>
            contacting the
            <br/>
            nucleic acid with a rare-cutting endonuclease to introduce the frameshift
            <br/>
            mutation. The
            <br/>
            rare-cutting endonuclease can be a transcription activator-like effector
            <br/>
            endonuclease
            <br/>
            (TALE nuclease), a meganuclease, a zinc finger nuclease (ZFN), or a clustered
            <br/>
            regularly
            <br/>
            interspaced short palindromic repeats (CRISPR)/CRISPR-associated (Cas)
            <br/>
            nuclease
            <br/>
            reagent. The polypeptide encoded by the nucleic acid containing the frameshift
            <br/>
            mutation
            <br/>
            can have increased sulfur-containing amino acid content as compared to a
            <br/>
            corresponding
            <br/>
            wild type polypeptide. The nucleic acid can encode a soybean globulin
            <br/>
            polypeptide,
            <br/>
            where the frameshift mutation is within the sequence set forth in SEQ ID
            <br/>
            NO:94, or a
            <br/>
            sequence having at least 90% identity to SEQ ID NO:94. The polypeptide encoded
            <br/>
            by the
            <br/>
            nucleic acid containing the frameshift mutation can be a soybean globulin
            <br/>
            polypeptide
            <br/>
            that contains the amino acid sequence set forth in SEQ ID NO:95. The
            <br/>
            polypeptide
            <br/>
            encoded by the nucleic acid containing the frameshift mutation can have
            <br/>
            increased
            <br/>
            threonine content as compared to a corresponding wild type polypeptide. The
            <br/>
            nucleic acid
            <br/>
            can encode a wheat alpha gliadin polypeptide, where the frameshift mutation is
            <br/>
            within the
            <br/>
            sequence set forth in SEQ ID NO:96, or a sequence having at least 90% identity
            <br/>
            to SEQ
            <br/>
            ID NO:96. The polypeptide encoded by the nucleic acid containing the
            <br/>
            frameshift
            <br/>
            mutation can be a wheat alpha gliadin polypeptide that contains the amino acid
            <br/>
            sequence
            <br/>
            set forth in SEQ ID NO:97, or an amino acid sequence having at least 90%
            <br/>
            sequence
            <br/>
            identity to SEQ ID NO:97. The nucleic acid can encode a wheat high molecular
            <br/>
            weight
            <br/>
            glutenin polypeptide, where the frameshift mutation is within the sequence set
            <br/>
            forth in
            <br/>
            SEQ ID NO:70, or a sequence having at least 90% identity to SEQ ID NO:70. The
            <br/>
            frameshift mutation can encompass or be 3' to the nucleotide at position 171
            <br/>
            of SEQ ID
            <br/>
            NO:70. The polypeptide encoded by the nucleic acid containing the frameshift
            <br/>
            mutation
            <br/>
            can be a wheat high molecular weight glutenin polypeptide that contains the
            <br/>
            amino acid
            <br/>
            sequence set forth in SEQ ID NO:98, or an amino acid sequence having at least
            <br/>
            90%
            <br/>
            identity to SEQ ID NO:98. The polypeptide encoded by the nucleic acid
            <br/>
            containing the
            <br/>
            frameshift mutation can have increased lysine content as compared to a
            <br/>
            corresponding
            <br/>
            wild type polypeptide. The nucleic acid can encode a wheat high molecular
            <br/>
            weight
            <br/>
            glutenin polypeptide, where the frameshift mutation is within the sequence set
            <br/>
            forth in
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            4
            <br/>
            SEQ ID NO:70, or a sequence having at least 90% identity to SEQ ID NO:70. The
            <br/>
            frameshift mutation can encompass or be 3' to the nucleotide at position 348
            <br/>
            of SEQ ID
            <br/>
            NO:70. The polypeptide encoded by the nucleic acid containing the frameshift
            <br/>
            mutation
            <br/>
            can be a wheat high molecular weight glutenin polypeptide that contains the
            <br/>
            amino acid
            <br/>
            sequence set forth in SEQ ID NO:99, or an amino acid sequence having at least
            <br/>
            90%
            <br/>
            identity to SEQ ID NO:99. The method can further include introducing a second
            <br/>
            frameshift mutation into the nucleic acid encoding the polypeptide, where the
            <br/>
            frameshift
            <br/>
            mutations in combination result in a deletion or insertion of nucleotides, and
            <br/>
            where the
            <br/>
            size of the deletion or insertion is a multiple of 3.
            <br/>
            1()    In another aspect, this document features a method for generating a
            <br/>
            plant, plant
            <br/>
            part, or plant cell with altered levels of amino acids, where the method
            <br/>
            includes (a)
            <br/>
            contacting a plant, plant part, or plant cell with a rare-cutting endonuclease
            <br/>
            targeted to a
            <br/>
            sequence within an exon of a gene endogenous to the plant, plant part, or
            <br/>
            plant cell, such
            <br/>
            that the rare-cutting endonuclease generates a double strand break at or near
            <br/>
            the sequence
            <br/>
            to which it is targeted, and (b) selecting a plant, plant part, or plant cell
            <br/>
            that contains a
            <br/>
            frameshift mutation within the exon, wherein the plant, plant part, or plant
            <br/>
            cell has altered
            <br/>
            amino acid levels as compared to a control plant, plant part, or plant cell in
            <br/>
            which the
            <br/>
            frameshift mutation was not introduced. The method can further include growing
            <br/>
            a plant
            <br/>
            part or plant cell selected in step (b) into a plant. In some embodiments, the
            <br/>
            plant cell that
            <br/>
            .. is contacted in step (a) can be a protoplast. The method can include
            <br/>
            transforming the
            <br/>
            protoplast with a nucleic acid (e.g., an RNA, or a nucleic acid contained
            <br/>
            within a vector)
            <br/>
            encoding the rare-cutting endonuclease. In some embodiments, the plant part
            <br/>
            that is
            <br/>
            contacted in step (a) can be an immature embryo or embryogenic callus. The
            <br/>
            method can
            <br/>
            include transforming the embryo or embryogenic callus with a nucleic acid
            <br/>
            encoding the
            <br/>
            rare-cutting endonuclease. The transforming can include Agrobacterium-mediated
            <br/>
            transformation or biolistics. The rare-cutting endonuclease can be a
            <br/>
            transcription
            <br/>
            activator-like effector endonuclease (TALE nuclease), meganuclease, zinc
            <br/>
            finger nuclease
            <br/>
            (ZFN), or clustered regularly interspaced short palindromic repeats
            <br/>
            (CRISPR)/CRISPR-
            <br/>
            associated (Cas) nuclease reagent. In some embodiments, the method can further
            <br/>
            include
            <br/>
            culturing the protoplasts, immature embryos, or embryogenic calli to generate
            <br/>
            plant lines.
            <br/>
            The frameshift mutation can be in the coding sequence of the gene, or within
            <br/>
            the last exon
            <br/>
            of the gene. The frameshift can be introduced by homologous recombination with
            <br/>
            a user-
            <br/>
            supplied donor molecule. The frameshift mutation can be within a gene that
            <br/>
            encodes a
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            seed storage protein (e.g., gliadin, hordein, secalin, zein, kafirin, avenin,
            <br/>
            glycinin, or
            <br/>
            conglycinin). In some cases, the seed storage protein encoded by the gene
            <br/>
            containing the
            <br/>
            frameshift mutation can contain the amino acid sequence set forth in SEQ ID
            <br/>
            NO:95,
            <br/>
            SEQ ID NO:98, or SEQ ID NO:99, or an amino acid sequence having at least 90%
            <br/>
            5    identity to SEQ ID NO:95, SEQ ID NO:98, or SEQ ID NO:99. The frameshift
            <br/>
            mutation
            <br/>
            can be within a gene that encodes a protein expressed in leaf tissue (e.g.,
            <br/>
            ribulose-1,5-
            <br/>
            bisphosphate (RuBP) carboxylase/oxygenase (rubisco), translational elongation
            <br/>
            factor
            <br/>
            EF-1 alpha (EF1a), or ubiquitin).
            <br/>
            In another aspect, this document features a method for generating a plant,
            <br/>
            plant
            <br/>
            part, or plant cell with altered levels of amino acids, where the method
            <br/>
            includes (a)
            <br/>
            contacting a plant, plant part, or plant cell with a first rare-cutting
            <br/>
            endonuclease targeted
            <br/>
            to a sequence within a gene endogenous to the plant, plant part, or plant
            <br/>
            cell, such that the
            <br/>
            first rare-cutting endonuclease generates a double strand break at or near the
            <br/>
            sequence to
            <br/>
            which it is targeted, (b) selecting a plant, plant part, or plant cell that
            <br/>
            contains a first
            <br/>
            frameshift mutation within the gene, (c) contacting a plant, plant part or
            <br/>
            plant cell with a
            <br/>
            second rare-cutting endonuclease targeted to a sequence within the same gene
            <br/>
            as that to
            <br/>
            which the first rare-cutting endonuclease was targeted, such that the second
            <br/>
            rare-cutting
            <br/>
            endonuclease generates a double strand break at or near the sequence to which
            <br/>
            it is
            <br/>
            targeted, and (d) selecting a plant, plant part, or plant cell that contains a
            <br/>
            second mutation
            <br/>
            within the endogenous gene. In some embodiments, the plant cell that is
            <br/>
            contacted in step
            <br/>
            (a) or step (c) can be a protoplast. The method can include transforming the
            <br/>
            protoplast
            <br/>
            with a nucleic acid (e.g., an mRNA or a nucleic acid contained within a
            <br/>
            vector) encoding
            <br/>
            the first or second rare-cutting endonuclease. In some embodiments, the plant
            <br/>
            part that is
            <br/>
            contacted in step (a) or step (c) can be an immature embryo or embryogenic
            <br/>
            callus. The
            <br/>
            method can include transforming the embryo or embryogenic callus with a
            <br/>
            nucleic acid
            <br/>
            encoding the first or second rare-cutting endonuclease. The transforming can
            <br/>
            include
            <br/>
            Agrobacterium-mediated transformation or transformation by biolistics. The
            <br/>
            first or
            <br/>
            second rare-cutting endonuclease can be a TALE nuclease, meganuclease, ZFN, or
            <br/>
            <br/>
            CRISPR/Cas reagent. The method can further include culturing the protoplast,
            <br/>
            immature
            <br/>
            embryo, or embryogenic callus to generate a plant line. The first frameshift
            <br/>
            mutation can
            <br/>
            be introduced chronologically before the second mutation, and the second
            <br/>
            mutation can
            <br/>
            be introduced into a plant, plant part, or plant cell selected in step (b).
            <br/>
            Alternatively, the
            <br/>
            second mutation can be introduced chronologically before the first frameshift
            <br/>
            mutation,
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            6
            <br/>
            and the first frameshift mutation can be introduced into a plant, plant part,
            <br/>
            or plant cell
            <br/>
            selected in step (d). The method of claim 17, wherein the first frameshift
            <br/>
            mutation is
            <br/>
            within an exon of the gene. The second mutation can be is downstream of the
            <br/>
            first
            <br/>
            frameshift mutation. The second mutation can be a frameshift mutation that re-
            <br/>
            introduces
            <br/>
            the normal reading frame found in the wild type gene. The second mutation can
            <br/>
            inactivate
            <br/>
            splicing of introns downstream from the first frameshift mutation. The first
            <br/>
            frameshift
            <br/>
            mutation or the second mutation can be introduced by homologous recombination
            <br/>
            using a
            <br/>
            user-generated donor molecule. The first frameshift mutation and the second
            <br/>
            mutation can
            <br/>
            be introduced simultaneously by homologous recombination using a user-
            <br/>
            generated donor
            <br/>
            molecule, or by simultaneously delivering two or more rare-cutting
            <br/>
            endonucleases. The
            <br/>
            frameshift mutation can be within a gene that encodes a seed storage protein
            <br/>
            (e.g., gliadin,
            <br/>
            hordein, secalin, zein, kafirin, avenin, glycinin, or conglycinin). In some
            <br/>
            cases, the seed
            <br/>
            storage protein encoded by the gene containing the frameshift mutation can
            <br/>
            contain the
            <br/>
            amino acid sequence set forth in SEQ ID NO:95, SEQ ID NO:98, or SEQ ID NO:99,
            <br/>
            or
            <br/>
            an amino acid sequence having at least 90% identity to SEQ ID NO:95, SEQ ID
            <br/>
            NO:98,
            <br/>
            or SEQ ID NO:99. The frameshift mutation can be within a gene that encodes a
            <br/>
            protein
            <br/>
            expressed in leaf tissue (e.g., rubisco, EF la, or ubiquitin).
            <br/>
            In another aspect, this document features a plant, plant part, or plant cell
            <br/>
            with
            <br/>
            altered levels of amino acids, wherein the plant contains a frameshift
            <br/>
            mutation in an exon
            <br/>
            of a selected gene. The altered levels of amino acids can have at least a 0.1%
            <br/>
            increase or
            <br/>
            decrease in the content of one or more amino acids. The plant, plant part, or
            <br/>
            plant cell can
            <br/>
            contain a second frameshift mutation within the selected gene. The plant,
            <br/>
            plant part, or
            <br/>
            plant cell can contain a second mutation within an exon or intron of the
            <br/>
            selected gene.
            <br/>
            The second mutation can be a deletion, insertion, substitution, or inversion
            <br/>
            of nucleotides
            <br/>
            that are required for intron splicing. The plant, plant part, or plant cell
            <br/>
            can be a wheat,
            <br/>
            cassava, alfalfa, oat, corn, rice, sorghum, potato, tomato, soybean, or canola
            <br/>
            plant, plant
            <br/>
            part, or plant cell.
            <br/>
            In addition, this document features a method for generating plant, plant cell,
            <br/>
            or
            <br/>
            plant part having a frameshift mutation in at least one protein-coding
            <br/>
            sequence that is
            <br/>
            endogenous to the plant, plant cell, or plant part such that the plant, plant
            <br/>
            cell, or plant
            <br/>
            part has increased or decreased levels of one or more amino acids of interest
            <br/>
            as compared
            <br/>
            to a control plant, plant cell, or plant part that lacks the frameshift
            <br/>
            mutation. The
            <br/>
            frameshift can be introduced by a deletion of nucleotides, or an insertion of
            <br/>
            nucleotides.
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            7
            <br/>
            The deletion of nucleotides can be a length of -3(N) ¨ 1, where N is any whole
            <br/>
            number,
            <br/>
            including zero. Furthermore, the deletion of nucleotides can be a length of -
            <br/>
            3(N) ¨ 2,
            <br/>
            where N is any whole number, including zero. The insertion of nucleotides can
            <br/>
            be a
            <br/>
            length of +3(N) + 1, where N is any whole number, including 0. Furthermore,
            <br/>
            the
            <br/>
            insertion of nucleotides can be a length of +3(N) + 2, where N is any whole
            <br/>
            number,
            <br/>
            including 0. In some embodiments, the mutation can include a combination of an
            <br/>
            <br/>
            insertion and deletion which results in a final increase in the length of
            <br/>
            nucleotides with
            <br/>
            the cumulative length of +3(N) + 1 or +3(N) + 2 nucleotides, where N is any
            <br/>
            whole
            <br/>
            number, including 0. In some embodiments, the mutation can include a
            <br/>
            combination of an
            <br/>
            insertion and deletion which results in a final decrease in the length of
            <br/>
            nucleotides with
            <br/>
            the cumulative length of -3(N) ¨ 1 or -3(N) ¨ 2 nucleotides, where N is any
            <br/>
            whole number
            <br/>
            including 0. The frameshift mutation can occur at a target sequence anywhere
            <br/>
            between
            <br/>
            the start codon and stop codon of a protein-coding gene that does not contain
            <br/>
            introns. The
            <br/>
            frameshift mutation can be at a target sequence within the last exon of a
            <br/>
            protein-coding
            <br/>
            gene. The mutation can be at a target sequence within the second to last exon
            <br/>
            of a protein-
            <br/>
            coding gene. The mutation can be at a target sequence within any exon of a
            <br/>
            protein-
            <br/>
            coding gene.
            <br/>
            In another aspect, this document features a method for generating a plant,
            <br/>
            plant
            <br/>
            cell, or plant part having an additional mutation downstream of a frameshift
            <br/>
            mutation,
            <br/>
            such that the a plant, plant cell, or plant part has increased expression of
            <br/>
            the protein-
            <br/>
            coding sequence containing the frameshift as compared to a control plant,
            <br/>
            plant cell, or
            <br/>
            plant part that does not contain the additional mutation, but contains the
            <br/>
            upstream
            <br/>
            frameshift mutation. The mutation can include a deletion of one or more
            <br/>
            nucleotides, an
            <br/>
            insertion of one or more nucleotides, a substitution of one or more
            <br/>
            nucleotides, or an
            <br/>
            inversion of sequence. In some embodiments, the mutation can include a
            <br/>
            combination of
            <br/>
            two or more of: deletion of one or more nucleotides, inversion of one or more
            <br/>
            nucleotides, insertion of one or more nucleotides, and substitution of one or
            <br/>
            more
            <br/>
            nucleotides within an allele. The mutation can result in the inactivation of
            <br/>
            intron splicing
            <br/>
            of one or more introns downstream of the stop codon introduced by the
            <br/>
            frameshift. The
            <br/>
            plant, plant cell, or plant part can have increased levels of gene expression
            <br/>
            of the protein-
            <br/>
            coding sequence containing the frameshift mutation, as compared to a plant,
            <br/>
            plant cell,
            <br/>
            plant part that does not contain the mutation, but contains the frameshift
            <br/>
            mutation.
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            8
            <br/>
            In still another aspect, this document features a plant, plant cell, or plant
            <br/>
            part
            <br/>
            having two frameshift mutations such that the plant, plant cell, or plant part
            <br/>
            has increased
            <br/>
            levels of the modified protein as compared to a control plant, plant cell, or
            <br/>
            plant part that
            <br/>
            does not contain the two frameshift mutations. In another aspect, this
            <br/>
            document features a
            <br/>
            plant, plant cell, or plant part having an additional mutation downstream of a
            <br/>
            frameshift
            <br/>
            mutation, such that the a plant, plant cell, or plant part has increased
            <br/>
            expression of the
            <br/>
            protein-coding sequence containing the frameshift as compared to a control
            <br/>
            plant, plant
            <br/>
            cell, or plant part that does not contain the additional mutation, but
            <br/>
            contains the upstream
            <br/>
            frameshift mutation.
            <br/>
            Unless otherwise defined, all technical and scientific terms used herein have
            <br/>
            the
            <br/>
            same meaning as commonly understood by one of ordinary skill in the art to
            <br/>
            which this
            <br/>
            invention pertains. Although methods and materials similar or equivalent to
            <br/>
            those
            <br/>
            described herein can be used to practice the invention, suitable methods and
            <br/>
            materials are
            <br/>
            described below. All publications, patent applications, patents, and other
            <br/>
            references
            <br/>
            mentioned herein are incorporated by reference in their entirety. In case of
            <br/>
            conflict, the
            <br/>
            present specification, including definitions, will control. In addition, the
            <br/>
            materials,
            <br/>
            methods, and examples are illustrative only and not intended to be limiting.
            <br/>
            The details of one or more embodiments of the invention are set forth in the
            <br/>
            accompanying drawings and the description below. Other features, objects, and
            <br/>
            advantages of the invention will be apparent from the description and
            <br/>
            drawings, and from
            <br/>
            the claims.
            <br/>
            DESCRIPTION OF DRAWINGS
            <br/>
            FIG. 1 is a diagram illustrating an approach for altering the amino acid
            <br/>
            content of
            <br/>
            a protein of interest. Step 1 involves the in silico analysis of all reading
            <br/>
            frames of a gene
            <br/>
            of interest to determine which reading frame has the highest level of the
            <br/>
            desired amino
            <br/>
            acid of interest. After finding the location of a desired reading frame, Step
            <br/>
            2 involves the
            <br/>
            design and delivery of a sequence-specific nuclease for creating a controlled
            <br/>
            frameshift
            <br/>
            mutation. In the example shown in Step 1, the reading frame with the highest
            <br/>
            level of the
            <br/>
            amino acid of interest is -1. Therefore, the size of nuclease-mediated
            <br/>
            deletion can be -
            <br/>
            3(N) ¨ 1, where N is any whole number, including 0. Notably, the mutation can
            <br/>
            also be an
            <br/>
            insertion with the size of +3(N) + 2, where N is any whole number including 0.
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            9
            <br/>
            FIG. 2 the genomic sequence encoding the soybean seed storage protein, Gy4
            <br/>
            (Glyma10g04280; SEQ ID NO:1). Upper case letters indicate exon sequences, and
            <br/>
            lower
            <br/>
            case letters indicate intron sequences. There are four exons and three introns
            <br/>
            within the
            <br/>
            Gy4 gene.
            <br/>
            FIGS. 3A and 3B illustrate a process for finding an alternative reading frame
            <br/>
            with
            <br/>
            high methionine and lysine codons. The figures show the Glycine max Gy4 exon 1
            <br/>
            (SEQ
            <br/>
            ID NO:2; FIG. 3A), exon 2 (SEQ ID NO:12; FIG. 3A), exon 3 (SEQ ID NO:20; FIG.
            <br/>
            3B), and exon 4 (SEQ ID NO:40; FIG. 3B) sequences, followed by the three
            <br/>
            translated
            <br/>
            frames for each exon. Underlined letters within the -1 frame of exon 3
            <br/>
            indicate the region
            <br/>
            with the highest level of methionine and lysine. Underlined letters within the
            <br/>
            exon 3
            <br/>
            sequence (SEQ ID NO:20) indicate the binding site of a TALE nuclease designed
            <br/>
            to
            <br/>
            introduce the desired -3(N) ¨ 1 or +3(N) + 2 frameshift mutation.
            <br/>
            FIG. 4 is an example of the amino acid sequence of Gy4 before a frameshift
            <br/>
            mutation (&gt;Gy4 wild type; left panel; SEQ ID NO:55) and after a frameshift
            <br/>
            mutation
            <br/>
            (&gt;Gy4; right panel; -1 frameshift within exon 3; early stop codon at the end
            <br/>
            of exon 3;
            <br/>
            SEQ ID NO:56). The methionine and cysteine content increases from 1.5% to
            <br/>
            4.1%, and
            <br/>
            the lysine content increases from 5% to 9.1%. Alternating normal font and
            <br/>
            italics indicate
            <br/>
            the different exons that encode the amino acids. The first 23 letters (bold)
            <br/>
            indicate the
            <br/>
            signal sequence. Methionine and cysteine amino acids are bold and underlined.
            <br/>
            FIG. 5 is an illustration of an approach to increase protein expression and
            <br/>
            stability. After the first frameshift is introduced using transcription
            <br/>
            activator-like effector
            <br/>
            endonuclease (TALE nuclease) 1, the mRNA transcript may be subjected to
            <br/>
            nonsense-
            <br/>
            mediated decay (top). To prevent nonsense-mediated decay, and to increase
            <br/>
            protein
            <br/>
            stability, a second TALE nuclease 2 can be designed to re-introduce the wild
            <br/>
            type reading
            <br/>
            frame after the codons of interest (bottom).
            <br/>
            FIG. 6 shows the amino acid sequence of Gy4 (&gt;Gy4 wild type; left panel; SEQ
            <br/>
            ID NO:55) and the sequence of Gy4 after the introduction of two frameshift
            <br/>
            mutations as
            <br/>
            illustrated in FIG. 5 (&gt;Gy4; right panel; -1 frameshift within exon 3;
            <br/>
            frameshift at the end
            <br/>
            of exon 3 to restore original frame; SEQ ID NO:57). The methionine and
            <br/>
            cysteine content
            <br/>
            increases from 1.5% to 3.3%, and the lysine content increases from 5% to 7.2%.
            <br/>
            FIG. 7 is an illustration of an approach to circumvent nonsense-mediated decay
            <br/>
            in
            <br/>
            genes with premature stop codons. After the first frameshift is introduced
            <br/>
            using TALE
            <br/>
            nuclease 1 (top), the mRNA transcript may be subjected to nonsense-mediated
            <br/>
            decay. To
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            prevent nonsense-mediated decay, a second TALE nuclease (TALE nuclease 2) is
            <br/>
            designed to mutate essential nucleotides involved in splicing (bottom).
            <br/>
            FIG. 8 illustrates a process for finding an alternative reading frame with
            <br/>
            high
            <br/>
            threonine codons. A representative Triticum aestivum alpha gliadin coding
            <br/>
            sequence
            <br/>
            5    (GENBANKO JN831386.1; SEQ ID NO:58) is followed by the three translated
            <br/>
            reading
            <br/>
            frames. Underlined letters within the -2 frame indicate the region with the
            <br/>
            highest level of
            <br/>
            threonine amino acids. Underlined letters in the alpha gliadin coding sequence
            <br/>
            indicate
            <br/>
            the binding site of a TALE nuclease designed to introduce the desired -3(N) ¨
            <br/>
            2 or +3(N)
            <br/>
            + 1 frameshift mutation.
            <br/>
            10    FIG. 9 is an example of the amino acid sequence of a WT alpha gliadin
            <br/>
            protein
            <br/>
            (&gt;Triticum aestivum clone 1-8 alpha gliadin (gli-2) gene, translated cds; left
            <br/>
            panel;
            <br/>
            GENBANKO JN831386.1; SEQ ID NO: 68) and an alpha gliadin protein where a -2
            <br/>
            frameshift occurs in the coding sequence near the start codon (&gt;Triticum
            <br/>
            aestivum clone
            <br/>
            1-8 alpha gliadin (gli-2) gene, translated cds; right panel; -2 frameshift
            <br/>
            mutation at the
            <br/>
            begining of the coding sequence; SEQ ID NO:69). The resulting protein has
            <br/>
            increased
            <br/>
            threonine and lysine content.
            <br/>
            FIGS. 10A and 10B illustrate a process for finding an alternative reading
            <br/>
            frame
            <br/>
            with high threonine and lysine codons. A representative Triticum aestivum
            <br/>
            glutenin
            <br/>
            coding sequence (FIG. 10A, Triticum aestivum Glu-1D-id gene for high molecular
            <br/>
            weight glutenin subunit 5; GENBANKO X12928.5; SEQ ID NO:70), followed by the
            <br/>
            three translated reading frames (FIG. 10B). Underlined letters within the -1
            <br/>
            and -2 frames
            <br/>
            indicate the regions with the highest level of lysine and threonine amino
            <br/>
            acids,
            <br/>
            respectively.
            <br/>
            FIG. 11 is an example of the amino acid sequence of a WT glutenin protein
            <br/>
            (&gt;Triticum aestivum Glu-1D-id gene for high molecular weight glutenin subunit
            <br/>
            5
            <br/>
            translated CDS; GENBANKO X12928.5; SEQ ID NO:90) and a glutenin protein with a
            <br/>
            -
            <br/>
            2 frameshift in the coding sequence near the start codon (&gt;Triticum aestivum
            <br/>
            Glu-1D-id
            <br/>
            gene for high molecular weight glutenin subunit 5 translated CDS; -2
            <br/>
            frameshift at the
            <br/>
            start of the coding sequence; SEQ ID ON:91). The resulting protein has
            <br/>
            increased
            <br/>
            threonine lysine content, relative to the wild type protein. Also shown is the
            <br/>
            amino acid
            <br/>
            sequence of a glutenin protein with a -1 frameshift (&gt; Triticum aestivum Glu-
            <br/>
            1D- ld gene
            <br/>
            for high molecular weight glutenin subunit 5 translated CDS; -1 frameshift at
            <br/>
            the 5' end
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            11
            <br/>
            of the coding sequence; SEQ ID NO:92). The resulting protein has increased
            <br/>
            levels of
            <br/>
            threonine and lysine compared to the wild type protein.
            <br/>
            DETAILED DESCRIPTION
            <br/>
            Cereal and legume crops have limited levels of essential amino acids. For
            <br/>
            example, legumes, including soybean, have limited levels of methionine (Met),
            <br/>
            while
            <br/>
            cereal crops, including barley, corn, sorghum, and wheat, have limited levels
            <br/>
            of lysine
            <br/>
            (Lys) and threonine (Thr) (see, e.g., Galili et al., Biol Chem, 386: 817-831,
            <br/>
            2005; Swine
            <br/>
            Nutrition (Lewis and Southern, Eds.), pp. 131-150, CRC Press, Boca Raton, FL,
            <br/>
            2014).
            <br/>
            .. Efforts to improve the Lys and/or Met amino acid content in cereal and
            <br/>
            legume crops
            <br/>
            typically have utilized one of two approaches ¨ classical breeding and genetic
            <br/>
            engineering, both of which have met with limited success. Challenges of
            <br/>
            classical
            <br/>
            breeding include (1) the need to specifically increase Lys and/or Met content
            <br/>
            in seeds but
            <br/>
            not vegetative tissues, due to deleterious effects on plant growth (Bright et
            <br/>
            al., Biochem
            <br/>
            Genet, 20: 229-243, 1982; Ghislain et al., Plant J, 8:733-743, 1995), and (2)
            <br/>
            the need to
            <br/>
            incorporate Lys and/or Met within the major seed storage proteins (Ufaz and
            <br/>
            Galili, Plant
            <br/>
            Physiol, 100: 1157-1163, 2008). Genetic engineering can alleviate such
            <br/>
            challenges. For
            <br/>
            example, genetic engineering can use seed-specific promotors to express genes
            <br/>
            with high
            <br/>
            levels of Lys or Met, or to express RNA or protein that leads to increased
            <br/>
            levels of Lys or
            <br/>
            Met. A strong understanding of amino acid metabolic pathways is required for
            <br/>
            such
            <br/>
            genetic engineering, however. Further, whereas many genetic engineering
            <br/>
            approaches
            <br/>
            have resulted in increased levels of Met or Lys, they also have been
            <br/>
            associated with
            <br/>
            abnormal and undesired plant phenotypes (Zeh et al., Plant Physiol, 127: 792-
            <br/>
            802, 2001).
            <br/>
            Examples of genetic engineering approaches to improve Lys or Met content have
            <br/>
            included seed-specific expression of a feedback-insensitive dihydropicolinate
            <br/>
            synthase
            <br/>
            enzyme of Lys synthesis (Zhu et al., Plant Cell, 15: 845-853, 2003),
            <br/>
            suppression of the
            <br/>
            Lys catabolism genes lysine ketoglutarate reductase/saccharopine dehydrogenase
            <br/>
            (Reyes
            <br/>
            et al., Plant Mol Biol, 69: 81-89, 2009), RNAi-mediated knockdown of low Lys
            <br/>
            containing zein genes (Huang et al., J Agric Food Chem, 52: 1958-1964, 2004),
            <br/>
            .. overexpression of the Met biosynthesis pathway gene cystathionine gamma-
            <br/>
            synthase
            <br/>
            (Kim et al., Plant Physiol, 128: 95-107, 2002), RNAi-mediated knockdown of
            <br/>
            threonine
            <br/>
            synthase (Zeh et al., Plant Physiol, 127: 792-802, 2001), and knockdown of the
            <br/>
            Met
            <br/>
            catabolic enzyme SAM synthase (Goto et al., Genes Genet Syst, 77: 89-95,
            <br/>
            2002).
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            12
            <br/>
            The methods provided herein include the use of tools for precise genome
            <br/>
            engineering (e.g., sequence-specific nucleases and donor molecules), and
            <br/>
            provide a novel
            <br/>
            approach for modulating amino acid content in crop plants and proteins. As
            <br/>
            used herein,
            <br/>
            the terms "amino acid levels" and "amino acid content" refer to the percentage
            <br/>
            of a
            <br/>
            specific amino acid among total amino acids. When referring to a plant, plant
            <br/>
            part, or
            <br/>
            plant cell, "content" or "level" refers to the number of specific amino acids
            <br/>
            divided by the
            <br/>
            total number of amino acids within the plant, plant part, or plant cell. For
            <br/>
            example, a
            <br/>
            soybean seed with 1% methionine refers to a seed that contains 1 methionine
            <br/>
            for every 99
            <br/>
            non-methionine amino acids, over the total population of amino acids.
            <br/>
            "Content" or
            <br/>
            1()    "level" also can refer to the percentage of a specific amino acid
            <br/>
            within a protein. For
            <br/>
            example, a protein with 1% methionine refers to a protein that contains 1
            <br/>
            methionine for
            <br/>
            every 99 non-methionine amino acids, over the total number of amino acids of
            <br/>
            the
            <br/>
            protein.
            <br/>
            The terms "altered" and "modulated," as used herein with regard to amino acid
            <br/>
            levels or amino acid content, refer to a change in the relative amount of one
            <br/>
            or more
            <br/>
            particular amino acids within a protein, plant, plant part, or plant cell,
            <br/>
            where the change is
            <br/>
            an increase or decrease of at least 0.1% (e.g., at least 0.25%, 0.5%, 1%, 5%,
            <br/>
            10%, 0.1 to
            <br/>
            0.5%, 0.5 to 1%, 1 to 3%, 3 to 5%, 5 to 10%, or more than 10%), relative to
            <br/>
            the level or
            <br/>
            content of the particular amino acid(s) in a corresponding protein, plant,
            <br/>
            plant part, or
            <br/>
            plant cell that has not been modified according to the methods described
            <br/>
            herein. For
            <br/>
            example, a modified soybean seed with 2% methionine levels has an altered
            <br/>
            level of
            <br/>
            amino acids compared to an unmodified soybean seed containing 1% methionine.
            <br/>
            The
            <br/>
            modified soybean seed has an increased methionine content of 1% compared to an
            <br/>
            <br/>
            unmodified soybean seed.
            <br/>
            The methods provided herein can include, for example, contacting a plant,
            <br/>
            plant
            <br/>
            part, or plant cell with a rare-cutting endonuclease targeted to a sequence
            <br/>
            within an exon
            <br/>
            of a gene endogenous to the plant, plant part, or plant cell (e.g., a gene
            <br/>
            encoding a seed
            <br/>
            storage protein, or a protein expressed in a particular tissue, such as
            <br/>
            leaves), such that the
            <br/>
            rare-cutting endonuclease generates a double strand break at or near the
            <br/>
            sequence to
            <br/>
            which it is targeted, and then selecting a plant, plant part, or plant cell
            <br/>
            that contains a
            <br/>
            frameshift mutation within the exon. The frameshift of interest can be
            <br/>
            predetermined
            <br/>
            according to the methods described herein, which can include, for example,
            <br/>
            determining
            <br/>
            which reading frame of the exon contains the desired (e.g., greatest) level of
            <br/>
            one or more
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            13
            <br/>
            particular amino acids (e.g., essential amino acids, including histidine,
            <br/>
            isoleucine, leucine,
            <br/>
            lysine, methionine, phenylalanine, threonine, tryptophan, and valine). Methods
            <br/>
            for
            <br/>
            determining whether a plant, plant part, or plant cell contains a frameshift
            <br/>
            mutation in a
            <br/>
            particular gene include those well known in the art.
            <br/>
            In some embodiments, the methods provided herein further can include
            <br/>
            contacting
            <br/>
            a plant, plant part or plant cell with a second rare-cutting endonuclease
            <br/>
            targeted to a
            <br/>
            sequence within the same gene as that to which the first rare-cutting
            <br/>
            endonuclease was
            <br/>
            targeted, such that the second rare-cutting endonuclease generates a double
            <br/>
            strand break
            <br/>
            at or near the sequence to which it is targeted, and then selecting a plant,
            <br/>
            plant part, or
            <br/>
            plant cell that contains a second mutation within the endogenous gene. The
            <br/>
            first and
            <br/>
            second mutations can be generated in either order, such that a plant, plant
            <br/>
            part, or plant
            <br/>
            identified as having the first frameshift mutation can be subsequently be
            <br/>
            contacted with
            <br/>
            the second rare-cutting endonuclease, or a plant, plant part, or plant cell
            <br/>
            identified as
            <br/>
            containing the second mutation can subsequently be contacted with the first
            <br/>
            rare-cutting
            <br/>
            endonuclease. In some cases, the methods provided herein can include
            <br/>
            simultaneously
            <br/>
            delivering two or more rare-cutting endonucleases, such that the first and
            <br/>
            second
            <br/>
            mutations are generated at essentially the same time. The second mutation can
            <br/>
            be
            <br/>
            upstream or downstream from the first frameshift mutation. In some cases, the
            <br/>
            second
            <br/>
            mutation can be a frameshift that re-introduces the normal reading frame that
            <br/>
            is found in
            <br/>
            the wild type gene, or the second mutation can inactivate splicing of introns
            <br/>
            downstream
            <br/>
            from the first frameshift mutation.
            <br/>
            The plant cells that are contacted with a rare-cutting endonuclease can be,
            <br/>
            for
            <br/>
            example, protoplasts. Plant parts that can be contacted with a rare-cutting
            <br/>
            endonuclease
            <br/>
            include, without limitation, immature embryos, cotyledons, leaves, floral
            <br/>
            organs, roots,
            <br/>
            stems, or embryonic calli. The contacting can include, for example,
            <br/>
            transformation with a
            <br/>
            nucleic acid (e.g., a DNA or RNA, including DNA or RNA within a vector)
            <br/>
            encoding the
            <br/>
            rare-cutting endonuclease. In some embodiments, for example, a plant, plant
            <br/>
            part, or plant
            <br/>
            cell can be transformed with an mRNA encoding the rare-cutting endonuclease.
            <br/>
            Any
            <br/>
            suitable method of transformation can be used, including, without limitation,
            <br/>
            Agrobacterium-mediated transformation, polyethylene glycol (PEG) mediated
            <br/>
            transformation, electroporation, calcium phosphate mediated transformation,
            <br/>
            virus-
            <br/>
            mediated transformation, microinjection, laser mediated transformation,
            <br/>
            liposome
            <br/>
            mediated transformation, or techniques utilizing cell-penetrating peptides,
            <br/>
            silicon carbide
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            14
            <br/>
            fibers, or biolistics. The methods provided herein also may include culturing
            <br/>
            transformed
            <br/>
            protoplasts, immature embryos, or embryogenic calli to generate plant lines.
            <br/>
            In some cases, a frameshift mutation and/or a second mutation can be
            <br/>
            introduced
            <br/>
            by homologous recombination with an exogenous donor molecule (e.g., a donor
            <br/>
            molecule
            <br/>
            provided by the entity carrying out the method). Further, the first frameshift
            <br/>
            mutation and
            <br/>
            the second mutation can be introduced simultaneously by homologous
            <br/>
            recombination
            <br/>
            using a single donor molecule that includes both mutations.
            <br/>
            In some embodiments, when a plant part or plant cell has been identified as
            <br/>
            containing a desired frameshift mutation and/or a desired second mutation, the
            <br/>
            methods
            <br/>
            provided herein can further include growing the plant part or plant cell into
            <br/>
            a plant.
            <br/>
            It is to be noted that while the examples described herein focus on increasing
            <br/>
            Lys
            <br/>
            and/or Met levels in soybean or Lys and/or Thr levels in wheat, it is to be
            <br/>
            noted that this
            <br/>
            approach can be extended to modulating the content of other amino acids in
            <br/>
            additional
            <br/>
            crop species. For example, the methods provided herein can be used to modulate
            <br/>
            the
            <br/>
            levels of one or more essential amino acids (e.g., histidine, isoleucine,
            <br/>
            leucine, lysine,
            <br/>
            methionine, phenylalanine, threonine, tryptophan, and valine) in a crop
            <br/>
            species such as,
            <br/>
            without limitation, cassava, alfalfa, oat, corn, rice, sorghum, potato,
            <br/>
            tomato, or canola, as
            <br/>
            well as soybean or wheat.
            <br/>
            Soybean (Glycine max L. Men.) is an important source of protein for livestock
            <br/>
            production and is of growing importance as a protein source for human
            <br/>
            consumption.
            <br/>
            Although soybean has the highest protein content among seed crops, the protein
            <br/>
            quality is
            <br/>
            poor due to deficiencies in the content of the sulfur-containing amino acids,
            <br/>
            methionine
            <br/>
            and cysteine. Increasing the amount of methionine and cysteine in the amino
            <br/>
            acid profile
            <br/>
            of soybean meal would enhance its value for producers and consumers.
            <br/>
            Soybean 7S globulin (13-conglycinin) and 11S globulin (glycinin) are the two
            <br/>
            major protein components of the seed. These two major storage proteins in
            <br/>
            soybean seeds
            <br/>
            usually are identified by their sedimentation rates in sucrose gradients (Hill
            <br/>
            and
            <br/>
            Breidenbach, Plant Physiol, 53:747-751, 1974). The 11S protein (glycinin,
            <br/>
            legumin)
            <br/>
            consists of at least four acidic subunits and four basic subunits (Staswick et
            <br/>
            al. J Biol
            <br/>
            Chem, 256:8752-8755, 1981). These subunits are produced by the cleavage of
            <br/>
            precursor
            <br/>
            polypeptides that have been identified through in vitro translation and pulse-
            <br/>
            labeling
            <br/>
            experiments (Barton et al. J Biol Chem, 257:6089-6095, 1982). The 7S storage
            <br/>
            protein
            <br/>
            (conglycinin, vicilin) is a glycoprotein composed of a, a', and 13-subunits
            <br/>
            (Beachy et al, J
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            Mol App! Genet, 1:19-27, 1981). Together, the 7S and 11S storage proteins
            <br/>
            constitute
            <br/>
            about 70% of the total seed protein at maturity, and 30% to 40% of the mature
            <br/>
            seed
            <br/>
            weight. Other major proteins in soybean seeds include urease, lectin, and
            <br/>
            trypsin
            <br/>
            inhibitors.
            <br/>
            5     Wheat (Triticum aestivum) is one of the most-produced cereals worldwide,
            <br/>
            with
            <br/>
            an estimated annual production of 713 million tons (Food and Agricultural
            <br/>
            Organization
            <br/>
            of the United Nations (FAOSTAT), 2010 Crop Production Data, online at
            <br/>
            faostat.fao.org/site/567/DesktopDefault.aspx?PageID=567#ancor). Wheat grain is
            <br/>
            used to
            <br/>
            make flour for breads, cakes, pastas and biscuits, and to make beer and
            <br/>
            biofuels. Gluten,
            <br/>
            10    the major protein component in wheat grains, is primarily composed of
            <br/>
            gliadins (alcohol-
            <br/>
            water soluble) and glutenins (insoluble). The gliadins can be divided into
            <br/>
            three subclasses
            <br/>
            of proteins: a-,y-, and co-gliadins. The genes encoding gliadin proteins are
            <br/>
            present in
            <br/>
            tightly-linked clusters within the Gil-] loci (y- and co-gliadins), Gli-2 loci
            <br/>
            (a-gliadins),
            <br/>
            and Gli-3 loci (co-gliadins). The Gil-1 loci are present on the short arm of
            <br/>
            the homologous
            <br/>
            15    group 1 chromosomes (Gil-A], Gil-B], and Gil-Di), whereas the Gli-2 loci
            <br/>
            are found on
            <br/>
            the short arm of chromosome 6 (Gli-A2, Gli-B2, and Gli-D2). The copy number of
            <br/>
            gliadin
            <br/>
            genes within hexaploid wheat genomes is estimated to be 25 to 150 copies for a-
            <br/>
            gliadins,
            <br/>
            15 to 18 copies for co-gliadins, and 17 to 39 copies for y-gliadins (Gil-
            <br/>
            Humanes et al.,
            <br/>
            Proc Natl Acad Sci USA, 107:17023-17028, 2012).
            <br/>
            As used herein, the terms "plant" and "plant part" refer to cells, tissues,
            <br/>
            organs,
            <br/>
            grains, and severed parts (e.g., roots, leaves, and flowers) that retain the
            <br/>
            distinguishing
            <br/>
            characteristics of the parent plant. "Seed" refers to any plant structure that
            <br/>
            is formed by
            <br/>
            continued differentiation of the ovule of the plant, following its normal
            <br/>
            maturation point,
            <br/>
            irrespective of whether it is formed in the presence or absence of
            <br/>
            fertilization and
            <br/>
            .. irrespective of whether or not the grain structure is fertile or infertile.
            <br/>
            In addition to soybean and wheat, crop plants that can be modified according
            <br/>
            to
            <br/>
            the methods provided herein include, without limitation,
            <br/>
            The term "gene" as used herein refers to a sequence of DNA that encodes a
            <br/>
            protein. A "gene" also refers to alleles of genes that are present at the same
            <br/>
            chromosomal
            <br/>
            position on the homologous chromosome. The term "genes" refers to more than
            <br/>
            one gene
            <br/>
            present within the same genome. A "wild type gene" is a naturally occurring
            <br/>
            gene (e.g., as
            <br/>
            found within naturally occurring plants) that encodes a protein, while a
            <br/>
            "mutant gene" or
            <br/>
            "modified gene" is a gene that has incurred one or more sequence changes,
            <br/>
            where the
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            16
            <br/>
            sequence changes result in the loss or modification of amino acids within the
            <br/>
            translated
            <br/>
            protein, as compared to the wild type gene. Such a "mutant gene" or "modified
            <br/>
            gene" can
            <br/>
            include one or more mutations in a gene's nucleic acid sequence.
            <br/>
            A representative example of a naturally occurring soybean globulin nucleotide
            <br/>
            sequence is shown in FIG. 2 herein (from the glycinin Gy4 gene; SEQ ID NO:1),
            <br/>
            and a
            <br/>
            representative example of a naturally occurring soybean globulin amino acid
            <br/>
            sequence is
            <br/>
            shown in FIG. 4 herein (encoded by Gy4; SEQ ID NO:55). The soybean plants,
            <br/>
            cells,
            <br/>
            plant parts, seeds, and progeny thereof that are provided herein can have one
            <br/>
            or more
            <br/>
            mutations in one or more endogenous globulin gene(s) (e.g., the Gy4 gene),
            <br/>
            such that
            <br/>
            .. amino acid content of the globulin protein is altered compared to a WT
            <br/>
            globulin protein.
            <br/>
            Thus, in some cases, the soybean plants, plant parts, plant cells, seeds, and
            <br/>
            progeny can
            <br/>
            exhibit altered overall levels of amino acids.
            <br/>
            A representative example of a naturally occurring wheat alpha gliadin
            <br/>
            nucleotide
            <br/>
            sequence is shown in FIG. 8 herein (SEQ ID NO:58), and a representative
            <br/>
            example of a
            <br/>
            naturally occurring wheat alpha gliadin amino acid sequence is shown in FIG. 9
            <br/>
            herein
            <br/>
            (SEQ ID NO:68). The wheat plants, cells, plant parts, seeds, and progeny
            <br/>
            thereof that are
            <br/>
            provided herein can have one or more mutations in one or more endogenous alpha
            <br/>
            gliadin
            <br/>
            gene(s), such that the amino acid content of the alpha gliadin protein is
            <br/>
            altered compared
            <br/>
            to a WT alpha gliadin protein. Thus, in some cases, the wheat plants, plant
            <br/>
            parts, plant
            <br/>
            cells, seeds, and progeny can exhibit altered overall levels of amino acids.
            <br/>
            A representative example of a naturally occurring wheat glutenin nucleotide
            <br/>
            sequence is shown in FIG. 10A herein (SEQ ID NO:70), and a representative
            <br/>
            example of
            <br/>
            a naturally occurring wheat glutenin amino acid sequence is shown in FIG. 11
            <br/>
            herein
            <br/>
            (SEQ ID NO:90). The wheat plants, cells, plant parts, seeds, and progeny
            <br/>
            thereof that are
            <br/>
            provided herein can have one or more mutations in one or more endogenous
            <br/>
            glutenin
            <br/>
            gene(s), such that amino acid content of the glutenin protein is altered
            <br/>
            compared to a WT
            <br/>
            alpha gliadin protein. Further, in some cases, the wheat plants, plant parts,
            <br/>
            plant cells,
            <br/>
            seeds, and progeny can exhibit altered overall levels of amino acids.
            <br/>
            The term "rare-cutting endonuclease" as used herein refers to a natural or
            <br/>
            engineered protein having endonuclease activity directed to a nucleic acid
            <br/>
            sequence with
            <br/>
            a recognition sequence (target sequence) that typically is about 12 to 40 bp
            <br/>
            in length (e.g.,
            <br/>
            14-40, 15-36, or 16-32 bp in length). Several rare-cutting endonucleases cause
            <br/>
            cleavage
            <br/>
            inside their recognition site, leaving 4 nt staggered cuts with 3'0H or 5'0H
            <br/>
            overhangs.
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            17
            <br/>
            These rare-cutting endonucleases may be meganucleases, such as wild type or
            <br/>
            variant
            <br/>
            proteins of homing endonucleases, more particularly those belonging to the
            <br/>
            dodecapeptide family (LAGLIDADG (SEQ ID NO:93); see, WO 2004/067736). In some
            <br/>
            embodiments, a rare-cutting endonuclease can be a fusion protein containing a
            <br/>
            DNA
            <br/>
            binding domain and a catalytic domain with cleavage activity. TALE nucleases
            <br/>
            and zinc-
            <br/>
            finger-nucleases (ZFNs) are examples of fusions of DNA binding domains with
            <br/>
            the
            <br/>
            catalytic domain of the endonuclease Fokl. For a review of rare-cutting
            <br/>
            endonucleases,
            <br/>
            see Baker, Nature Methods 9:23-26, 2012.
            <br/>
            Transcription activator-like (TAL) effectors are found in plant pathogenic
            <br/>
            bacteria
            <br/>
            in the genus Xanthomonas . These proteins play important roles in disease, or
            <br/>
            trigger
            <br/>
            defense, by binding host DNA and activating effector-specific host genes (see,
            <br/>
            e.g., Gu et
            <br/>
            al., Nature, 435:1122-1125, 2005; Yang et al., Proc Natl Acad Sci USA,
            <br/>
            103:10503-
            <br/>
            10508, 2006; Kay et al. Science, 318:648-651, 2007; Sugio et al., Proc Natl
            <br/>
            Acad Sci
            <br/>
            USA, 104:10720-10725, 2007; and Romer et al. Science, 318:645-648, 2007).
            <br/>
            Specificity
            <br/>
            depends on an effector-variable number of imperfect, typically 34 amino acid
            <br/>
            repeats
            <br/>
            (Schornack et al., J Plant Physiol, 163:256-272, 2006; and WO 2011/072246).
            <br/>
            Polymorphisms are present primarily at repeat positions 12 and 13, which are
            <br/>
            referred to
            <br/>
            herein as the repeat variable-diresidue (RVD).
            <br/>
            Another genome engineering tool uses RNA to direct DNA cleavage ¨ the
            <br/>
            clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-
            <br/>
            associated
            <br/>
            (Cas) system (see, e.g., Belahj et al., Plant Methods, 9:39, 2013). This
            <br/>
            system consist of a
            <br/>
            Cas9 endonuclease and a guide RNA (either a complex between a CRISPR RNA
            <br/>
            [crRNA] and trans-activating crRNA [tracrRNAL or a synthetic fusion between
            <br/>
            the 3'
            <br/>
            end of the crRNA and 5'end of the tracrRNA [sgRNA]). The guide RNA directs
            <br/>
            Cas9
            <br/>
            binding and DNA cleavage to homologous sequences that are adjacent to a proto-
            <br/>
            spacer
            <br/>
            adjacent motif (PAM; e.g., NGG for Cas9 from Streptococcus pyogenes). Once at
            <br/>
            the
            <br/>
            target DNA sequence, Cas9 generates a DNA double-strand break at a position
            <br/>
            three
            <br/>
            nucleotides from the 3' end of the crRNA targeting sequence. As there are
            <br/>
            several PAM
            <br/>
            motifs present in the nucleotide sequence of the globulin genes, the
            <br/>
            CRISPR/Cas system
            <br/>
            may be employed to introduce mutations within the globulin alleles within
            <br/>
            soybean plant
            <br/>
            cells in which the Cas9 endonuclease and the guide RNA are transfected and
            <br/>
            expressed.
            <br/>
            This approach can be used as an alternative to TALE nucleases in some
            <br/>
            instances, to
            <br/>
            obtain plants as described herein.
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            18
            <br/>
            "Mutagenesis" as used herein refers to processes in which mutations are
            <br/>
            introduced into a selected DNA sequence. Mutations induced by endonucleases
            <br/>
            generally
            <br/>
            are obtained by a double strand break, which results in insertion/deletion
            <br/>
            mutations
            <br/>
            ("indels") that can be detected by deep-sequencing analysis. Such mutations
            <br/>
            typically are
            <br/>
            deletions of several base pairs, and have the effect of introducing frameshift
            <br/>
            mutations. In
            <br/>
            the methods described herein, for example, mutagenesis occurs via double
            <br/>
            stranded DNA
            <br/>
            breaks made by TALE nucleases targeted to selected DNA sequences in a plant
            <br/>
            cell. Such
            <br/>
            mutagenesis results in "TALE nuclease-induced mutations" (e.g., TALE nuclease-
            <br/>
            induced
            <br/>
            knockouts). Following mutagenesis, plants can be regenerated from the treated
            <br/>
            cells using
            <br/>
            known techniques (e.g., planting seeds in accordance with conventional growing
            <br/>
            procedures, followed by self-pollination).
            <br/>
            In some embodiments, the proteins, plants, plant cells, plant parts, seeds,
            <br/>
            and
            <br/>
            progeny provided herein can be generated using a TALE nuclease system to make
            <br/>
            targeted mutations in one or more selected genes [e.g., one or more genes
            <br/>
            encoding seed
            <br/>
            storage proteins such as globulins, glycinin, or gliadin, or one or more genes
            <br/>
            expressed in
            <br/>
            leaf tissue, such as ribulose-1,5-bisphosphate carboxylase/oxygenase
            <br/>
            (rubisco),
            <br/>
            translational elongation factor EF-1 alpha (EF la), or ubiquitin]. Thus, this
            <br/>
            document
            <br/>
            provides materials and methods for using rare-cutting endonucleases (e.g.,
            <br/>
            TALE
            <br/>
            nucleases) to generate proteins, plants, and related products (e.g., seeds and
            <br/>
            plant parts)
            <br/>
            that can be used as protein sources with reduced levels of low sulfur-
            <br/>
            containing globulin
            <br/>
            proteins, due to mutations in globulin genes. Other sequence-specific
            <br/>
            nucleases also may
            <br/>
            be used to generate the desired plant material, including engineered homing
            <br/>
            endonucleases, ZFNs and RNA-guided endonucleases.
            <br/>
            In some cases, a mutation can be at a target sequence as set forth in a
            <br/>
            globulin
            <br/>
            coding sequence as set forth herein (e.g., a glycinin sequence as set forth
            <br/>
            SEQ ID NO:1, a
            <br/>
            gliadin sequence as set forth in SEQ ID NO:58, or a glutenin sequence as set
            <br/>
            forth in SEQ
            <br/>
            ID NO:70), or at a target sequence that is at least 90 percent (e.g., at least
            <br/>
            90 percent, at
            <br/>
            least 91 percent, at least 92 percent, at least 93 percent, at least 94
            <br/>
            percent, at least 95
            <br/>
            percent, at least 96 percent, at least 97 percent, at least 98 percent, at
            <br/>
            least 99 percent, 90
            <br/>
            to 95 percent, 95 to 98 percent, or 98 to 99 percent) identical to the
            <br/>
            sequence set forth in a
            <br/>
            sequence as set forth herein (e.g., SEQ ID NO:1, SEQ ID NO:58, or SEQ ID
            <br/>
            NO:70), or
            <br/>
            at a target sequence that, when translated, is at least 90 percent (e.g., at
            <br/>
            least 90 percent, at
            <br/>
            least 91 percent, at least 92 percent, at least 93 percent, at least 94
            <br/>
            percent, at least 95
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            19
            <br/>
            percent, at least 96 percent, at least 97 percent, at least 98 percent, at
            <br/>
            least 99 percent, 90
            <br/>
            to 95 percent, 95 to 98 percent, or 98 to 99 percent) identical to an amino
            <br/>
            acid sequence
            <br/>
            as set forth herein (e.g., SEQ ID NO:55, SEQ ID NO:68, or SEQ ID NO:90).
            <br/>
            The percent sequence identity between a particular nucleic acid or amino acid
            <br/>
            sequence and a sequence referenced by a particular sequence identification
            <br/>
            number is
            <br/>
            determined as follows. First, a nucleic acid or amino acid sequence is
            <br/>
            compared to the
            <br/>
            sequence set forth in a particular sequence identification number using the
            <br/>
            BLAST 2
            <br/>
            Sequences (Bl2seq) program from the stand-alone version of BLASTZ containing
            <br/>
            BLASTN version 2Ø14 and BLASTP version 2Ø14. This stand-alone version of
            <br/>
            BLASTZ can be obtained online at fr.com/blast or at ncbi.nlm.nih.gov.
            <br/>
            Instructions
            <br/>
            explaining how to use the Bl2seq program can be found in the readme file
            <br/>
            accompanying
            <br/>
            BLASTZ. Bl2seq performs a comparison between two sequences using either the
            <br/>
            BLASTN or BLASTP algorithm. BLASTN is used to compare nucleic acid sequences,
            <br/>
            while BLASTP is used to compare amino acid sequences. To compare two nucleic
            <br/>
            acid
            <br/>
            sequences, the options are set as follows: -i is set to a file containing the
            <br/>
            first nucleic acid
            <br/>
            sequence to be compared (e.g., C:\seql.txt); -j is set to a file containing
            <br/>
            the second
            <br/>
            nucleic acid sequence to be compared (e.g., C:\5eq2.txt); -p is set to blastn;
            <br/>
            -o is set to any
            <br/>
            desired file name (e.g., C:\output.txt); -q is set to -1; -r is set to 2; and
            <br/>
            all other options are
            <br/>
            left at their default setting. For example, the following command can be used
            <br/>
            to generate
            <br/>
            an output file containing a comparison between two sequences: C:\B12seq
            <br/>
            c:\seql.txt -j
            <br/>
            c:\seq2.txt -p blastn -o c:\output.txt -q -1 -r 2. To compare two amino acid
            <br/>
            sequences, the
            <br/>
            options of Bl2seq are set as follows: -i is set to a file containing the first
            <br/>
            amino acid
            <br/>
            sequence to be compared (e.g., C:\seql.txt); -j is set to a file containing
            <br/>
            the second amino
            <br/>
            acid sequence to be compared (e.g., C:\5eq2.txt); -p is set to blastp; -o is
            <br/>
            set to any desired
            <br/>
            file name (e.g., C:\output.txt); and all other options are left at their
            <br/>
            default setting. For
            <br/>
            example, the following command can be used to generate an output file
            <br/>
            containing a
            <br/>
            comparison between two amino acid sequences: C:\B12seq c:\seql.txt -j
            <br/>
            c:\seq2.txt -p
            <br/>
            blastp -o c:\output.txt. If the two compared sequences share homology, then
            <br/>
            the
            <br/>
            designated output file will present those regions of homology as aligned
            <br/>
            sequences. If the
            <br/>
            .. two compared sequences do not share homology, then the designated output
            <br/>
            file will not
            <br/>
            present aligned sequences.
            <br/>
            Once aligned, the number of matches is determined by counting the number of
            <br/>
            positions where an identical nucleotide or amino acid residue is presented in
            <br/>
            both
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            sequences. The percent sequence identity is determined by dividing the number
            <br/>
            of
            <br/>
            matches either by the length of the sequence set forth in the identified
            <br/>
            sequence (e.g.,
            <br/>
            SEQ ID NO:1), or by an articulated length (e.g., 100 consecutive nucleotides
            <br/>
            or amino
            <br/>
            acid residues from a sequence set forth in an identified sequence), followed
            <br/>
            by
            <br/>
            5    multiplying the resulting value by 100. For example, a nucleic acid
            <br/>
            sequence that has
            <br/>
            2500 matches when aligned with the sequence set forth in SEQ ID NO:1 is 96.2
            <br/>
            percent
            <br/>
            identical to the sequence set forth in SEQ ID NO:1 (i.e., 2500   2600 x 100 =
            <br/>
            96.2). It is
            <br/>
            noted that the percent sequence identity value is rounded to the nearest
            <br/>
            tenth. For
            <br/>
            example, 75.11, 75.12, 75.13, and 75.14 is rounded down to 75.1, while 75.15,
            <br/>
            75.16,
            <br/>
            10    75.17, 75.18, and 75.19 is rounded up to 75.2. It also is noted that the
            <br/>
            length value will
            <br/>
            always be an integer.
            <br/>
            Methods for selecting endogenous target sequences and generating TALE
            <br/>
            nucleases targeted to such sequences can be performed as described elsewhere.
            <br/>
            See, for
            <br/>
            example, PCT Publication No. WO 2011/072246, which is incorporated herein by
            <br/>
            15    reference in its entirety. In some embodiments, software that
            <br/>
            specifically identifies TALE
            <br/>
            nuclease recognition sites, such as TALE-NT 2.0 (Doyle et al., Nucl Acids Res,
            <br/>
            40:W117-
            <br/>
            122, 2012) can be used.
            <br/>
            This document therefore provides materials and methods for generating
            <br/>
            proteins,
            <br/>
            plants, plant parts, and plant cells with altered amino acid content as
            <br/>
            compared to a
            <br/>
            20    corresponding wild type protein, plant, plant part, or plant cell. In
            <br/>
            some embodiments, for
            <br/>
            example, a method as provided herein can include contacting a plant, plant
            <br/>
            part, or plant
            <br/>
            cell with a rare-cutting endonuclease (e.g., a TALE nuclease) targeted to a
            <br/>
            sequence
            <br/>
            within an exon of a gene endogenous to the plant, plant part, or plant cell,
            <br/>
            such that the
            <br/>
            rare-cutting endonuclease generates a double strand break at or near the
            <br/>
            sequence to
            <br/>
            which it is targeted; and then selecting a plant, plant part, or plant cell
            <br/>
            that contains a
            <br/>
            frameshift mutation within the exon, where the plant, plant part, or plant
            <br/>
            cell has an
            <br/>
            altered amino acid content as compared to a control plant, plant part, or
            <br/>
            plant cell in
            <br/>
            which the frameshift mutation was not introduced. In some cases, the method
            <br/>
            also can
            <br/>
            include evaluating alternate reading frames for the gene or the exon, to
            <br/>
            determine which
            <br/>
            reading frame would produce a protein having the desired amino acid content.
            <br/>
            In some embodiments, the materials and methods provided herein can be used to
            <br/>
            generate a Gy4 protein having increased sulfur-containing amino acid content,
            <br/>
            by
            <br/>
            introducing a mutation into a Gy4 genomic sequence. The mutation can be a
            <br/>
            frameshift
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            21
            <br/>
            mutation of the size -3(N) ¨ 1 or +3(N) + 2; such a frameshift within exon 3
            <br/>
            of a Gy4
            <br/>
            gene (SEQ ID NO:20), or within a sequence having at least 90% sequence
            <br/>
            identity to
            <br/>
            SEQ ID NO:20, can be particularly useful. In some cases, a frameshift mutation
            <br/>
            of the
            <br/>
            size -3(N) ¨ 1 or +3(N) + 2 can be introduced (e.g., using one or more TALE
            <br/>
            nucleases)
            <br/>
            within a segment of a Gy4 gene that contains the sequence
            <br/>
            TCGTGACAGTGGAAGGAGGTCTCAGCGTTATCAGCCCCA AGTGGCAAGAA
            <br/>
            (SEQ ID NO:94), or within a sequence having at least 90% identity to the
            <br/>
            sequence set
            <br/>
            forth in SEQ ID NO:94. In some cases, the frameshift mutation can result in
            <br/>
            production
            <br/>
            of a protein that contains the amino acid sequence set forth in SEQ ID NO:95,
            <br/>
            or an
            <br/>
            amino acid sequence having at least 90% identity to SEQ ID NO:95 (MKMKMKTKMM
            <br/>
            KMNKFPLTLLADQAMESVNKTRTRTKMKINLVLVDQAKESVNKTRTRTRTKMK
            <br/>
            MKINLARKSREWRSKKTQPRRPRQEEPRERGCETRNGVEENIC).
            <br/>
            In some embodiments, the materials and methods provided herein can be used to
            <br/>
            generate an alpha gliadin protein having increased threonine content, by
            <br/>
            introducing a
            <br/>
            mutation into an alpha gliadin genomic sequence. The mutation can be a
            <br/>
            frameshift
            <br/>
            mutation of the size -3(N) ¨ 2 or +3(N) + 1. In some cases, a frameshift
            <br/>
            mutation of the
            <br/>
            size -3(N) ¨ 2 or +3(N) + 1 can be introduced (e.g., using one or more TALE
            <br/>
            nucleases)
            <br/>
            within a segment of the alpha gliadin gene that includes the sequence
            <br/>
            ATGAAGACCTTTCTCATCCTTGC
            <br/>
            CCTCCGTGCTATTGTAGCAACCACCGCCACAATT (SEQ ID NO:96), or within a
            <br/>
            sequence having at least 90% identity to SEQ ID NO:96. In some cases, the
            <br/>
            frameshift
            <br/>
            mutation can result in production of a protein that contains the amino acid
            <br/>
            sequence set
            <br/>
            forth in SEQ ID NO:97, or an amino acid sequence having at least 90% identity
            <br/>
            to SEQ
            <br/>
            ID NO:97 (TGPG LCPASTTAPV).
            <br/>
            In some embodiments, the materials and methods provided herein can be used to
            <br/>
            generate a high molecular weight glutenin protein with increased threonine
            <br/>
            content, by
            <br/>
            introducing a mutation into a high molecular weight glutenin genomic sequence.
            <br/>
            The
            <br/>
            mutation can be a frameshift mutation of the size -3(N) ¨ 2 or +3(N) + 1. In
            <br/>
            some cases, a
            <br/>
            frameshift mutation of the size -3(N) ¨ 2 or +3(N) + 1 can be introduced
            <br/>
            (e.g., using one
            <br/>
            or more TALE nucleases) into a high molecular weight glutenin nucleotide
            <br/>
            sequence
            <br/>
            containing the sequence set forth in SEQ ID NO:70, or into a sequence having
            <br/>
            at least
            <br/>
            90% identity to SEQ ID NO:70. The frameshift can occur at any suitable
            <br/>
            position within
            <br/>
            the high molecular weight glutenin sequence; in some cases, the frameshift
            <br/>
            mutation can
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            22
            <br/>
            encompass or follow the nucleotide at position 171 of SEQ ID NO:70. In some
            <br/>
            cases, the
            <br/>
            frameshift mutation can result in in production of a high molecular weight
            <br/>
            glutenin
            <br/>
            protein containing the amino acid sequence set forth in SEQ ID NO:98, or an
            <br/>
            amino acid
            <br/>
            sequence with at least 90% identity to the sequence set forth in SEQ ID NO:98
            <br/>
            (TDRTRAAIRTRATRLLQLIPC).
            <br/>
            Further, the materials and methods provided herein can be used to generate a
            <br/>
            high
            <br/>
            molecular weight glutenin protein having increased lysine content, by
            <br/>
            introducing a
            <br/>
            mutation into a high molecular weight glutenin genomic sequence. The mutation
            <br/>
            can be a
            <br/>
            frameshift mutation of the size -3(N) ¨ 1 or +3(N) + 2. In some cases, a
            <br/>
            frameshift
            <br/>
            mutation of the size -3(N) ¨ 2 or +3(N) + 1 can be introduced (e.g., using one
            <br/>
            or more
            <br/>
            TALE nucleases) within a high molecular weight glutenin sequence as set forth
            <br/>
            in SEQ
            <br/>
            ID NO:70, or within a sequence having at least 90% identity to SEQ ID NO:70.
            <br/>
            The
            <br/>
            frameshift can be at any suitable position within the high molecular weight
            <br/>
            glutenin
            <br/>
            nucleotide sequence, and in some cases, the frameshift mutation can encompass
            <br/>
            or follow
            <br/>
            the nucleotide at position 348 of SEQ ID NO:70. In some cases, the frameshift
            <br/>
            mutation
            <br/>
            can result in production of a high molecular weight glutenin protein
            <br/>
            containing the amino
            <br/>
            acid sequence set forth in SEQ ID NO:99, or an amino acid sequence having at
            <br/>
            least 90%
            <br/>
            identity to SEQ ID NO:99 (LLCNSRDKGNQGTTQLLCSS).
            <br/>
            The invention will be further described in the following examples, which do
            <br/>
            not
            <br/>
            limit the scope of the invention described in the claims.
            <br/>
            EXAMPLES
            <br/>
            Example 1 ¨ Searching for alternative reading frames within the soybean
            <br/>
            glycinin Gy4
            <br/>
            gene that code for high levels of methionine and lysine amino acids
            <br/>
            To increase methionine and lysine content in soybean, the storage protein Gy4
            <br/>
            (Glymal0g04280) was targeted for modification. The amino acid sequence of the
            <br/>
            wild
            <br/>
            type Gy4 protein contains 1.5% methionine and cysteine residues (combined) and
            <br/>
            5%
            <br/>
            lysine residues. The genomic sequence of the wild type Gy4 gene includes four
            <br/>
            exons and
            <br/>
            three introns (SEQ ID NO:1; FIG. 2). The approach illustrated in FIG. 1 was
            <br/>
            followed to
            <br/>
            generate a modified Gy4 protein with higher levels of lysine and methionine.
            <br/>
            The first
            <br/>
            step involved searching for alternative reading frames within the Gy4 coding
            <br/>
            sequence
            <br/>
            that contain high levels of methionine and lysine codons. To this end, the
            <br/>
            four exon
            <br/>
            sequences were translated in all three reading frames (FIGS. 3A and 3B). As
            <br/>
            expected,
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            23
            <br/>
            numerous stop codons were found in the -1 and -2 frames. However, there were
            <br/>
            regions
            <br/>
            between two stop codons with high levels of methionine and lysine codons. In
            <br/>
            particular,
            <br/>
            there was a stretch of codons in the -1 frame of exon 3 that encode 10
            <br/>
            methionine and 22
            <br/>
            lysine residues, whereas the same nucleotides within the normal reading frame
            <br/>
            encode 0
            <br/>
            methionine and 8 lysine residues (FIG. 3B). If a frameshift mutation occurs
            <br/>
            within the
            <br/>
            wild type Gy4 gene at the start of the alternative reading frame containing
            <br/>
            high levels of
            <br/>
            methionine and lysine, then the resulting Gy4 protein will contain about 4.1%
            <br/>
            methionine
            <br/>
            and cysteine amino acids (combined level), and 9.1% lysine (FIG. 4). A list of
            <br/>
            changes to
            <br/>
            all essential amino acids is provided in TABLE 1.
            <br/>
            TABLE 1
            <br/>
            Percent of essential amino acids in Gy4 after introducing a -1 frameshift
            <br/>
            within exon 3
            <br/>
            Glycine max Gy4 (FIG. 4)
            <br/>
            -1 Frameshift                                           Change
            <br/>
            Essential                 % amino acid
            <br/>
            ( /0 of amino                                          from WT
            <br/>
            Amino Acid in WT protein
            <br/>
            acid)                                                       CYO
            <br/>
            His                            2.78        2.76             -0.02
            <br/>
            Ile                            3.90        4.14             0.25
            <br/>
            Leu                            6.86        7.73             0.87
            <br/>
            Met                            0.37        3.31             2.94
            <br/>
            Phe                            2.60        2.21             -0.39
            <br/>
            Thr                            3.71        5.52             1.81
            <br/>
            Trp                            1.11        0.83             -0.28
            <br/>
            Val                            6.49        4.70             -1.80
            <br/>
            Lys                            5.01        9.12             4.11
            <br/>
            Thus, to generate a Gy4 protein with increased sulfur-containing amino acid
            <br/>
            content, mutations are introduced into the Gy4 genomic sequence such that one
            <br/>
            or more
            <br/>
            frameshift mutations of the size -3(N) - 1 or +3(N) + 2 occur, particularly
            <br/>
            within exon 3
            <br/>
            (SEQ ID NO:20) or within a sequence having at least 90% identity to SEQ ID
            <br/>
            NO:20. In
            <br/>
            some cases, a TALE nuclease is used to introduce a frameshift mutation of the
            <br/>
            size -3(N)
            <br/>
            - 1 or +3(N) + 2 within Gy4 exon 3, where the mutation is within the sequence
            <br/>
            set forth
            <br/>
            in SEQ ID NO:94, or within a sequence having at least 90% identity to SEQ ID
            <br/>
            NO:94
            <br/>
            (TCGTGACAGTGGAA GGAGGTCTCAGCGTTATCAGCCCCAAGTGGCAAGAA).
            <br/>
            The frameshift mutation within Gy4 exon 3 may result in production of a Gy4
            <br/>
            protein
            <br/>
            containing the amino acid sequence set forth in SEQ ID NO:95, or an amino acid
            <br/>
            <br/>
            sequence having at least 90% identity to SEQ ID NO:95
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            24
            <br/>
            (MKMKMKTKMMKMNKFPLTLLADQAMESVNKTRTRTKMKINLV
            <br/>
            LVDQAKESVNKTRTRTRTKMKMKINLARKSREWRSKKTQPRRPRQEEPRERGCE
            <br/>
            TRNGVEENIC).
            <br/>
            Example 2 ¨ In silico design of sequence-specific nucleases for introducing a -
            <br/>
            1
            <br/>
            frameshift in exon 3 of Gy4
            <br/>
            Having identified a reading frame within Gy4 that codes for a high level of
            <br/>
            methionine and lysine amino acids, the next step is to design sequence-
            <br/>
            specific nucleases
            <br/>
            to introduce the appropriate -1 frameshift mutation. Ideally, the frameshift
            <br/>
            should occur
            <br/>
            upstream of the first codon of interest in the alternative reading frame.
            <br/>
            However, there are
            <br/>
            two restrictions to where the frameshift can occur. First, the frameshift must
            <br/>
            occur
            <br/>
            downstream of the stop codon within the frame of interest that precedes the
            <br/>
            codons of
            <br/>
            interest. Notably, the frameshift can occur within the stop codon, as long as
            <br/>
            the stop
            <br/>
            codon is disrupted during the process. The second restriction is that the
            <br/>
            downstream stop
            <br/>
            codon in the alternative frame of interest should ideally occur after the last
            <br/>
            intron. If a
            <br/>
            stop codon is created that is before intron sequences, then the mRNA
            <br/>
            transcript may be
            <br/>
            subject to nonsense-mediated decay. However, to circumvent nonsense-mediated
            <br/>
            decay,
            <br/>
            additional methods are described herein, including disruption of intron
            <br/>
            splicing through
            <br/>
            mutations, and restoration of the original reading frame.
            <br/>
            To introduce the appropriate frameshift within Gy4, TALE nuclease pairs are
            <br/>
            designed to recognize sequence within exon 3 upstream of the codons of
            <br/>
            interest in the -1
            <br/>
            frame (FIGS. 2 and 3). The desired deletion size should have a total length of
            <br/>
            -3(N) ¨ 1,
            <br/>
            where N is a whole number, including zero. The desired insertion size should
            <br/>
            have a total
            <br/>
            length of +3(N) + 2 where N is a whole number, including zero. Notably, the
            <br/>
            deletion size
            <br/>
            does not typically exceed ¨40 bp, as methionine codons may start to be
            <br/>
            deleted.
            <br/>
            Example 3 ¨ Activity of Gy4 TALE nuclease pairs at their endogenous target
            <br/>
            sites in
            <br/>
            soybean
            <br/>
            To assess the activity of Gy4 TALE nuclease pairs at their endogenous target
            <br/>
            sequences, TALE nucleases are transformed into soybean protoplasts, and target
            <br/>
            sites are
            <br/>
            surveyed two days post transformation for mutations introduced by NHEJ.
            <br/>
            Methods for
            <br/>
            DNA transformation into soybean protoplasts are performed as described
            <br/>
            elsewhere (Dhir
            <br/>
            et al., Plant Cell Rep, 10: 39-43, 1991). Briefly, 15 days after pollination,
            <br/>
            immature
            <br/>
            soybean seedpods are sterilized by washing them successively on 100% ethanol,
            <br/>
            50%
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            bleach, and then sterile distilled water. Seedpod and seed coat are removed to
            <br/>
            isolate
            <br/>
            immature seeds. Protoplasts are then isolated from immature cotyledons by
            <br/>
            enzyme
            <br/>
            digestion for 16 hours using protocols described elsewhere (Dhir et al.,
            <br/>
            supra).
            <br/>
            TALE nuclease-encoding plasmids are next introduced into soybean protoplasts
            <br/>
            5    by PEG-mediated transformation (Yoo et al., Nat Protoc, 2:1565-1572,
            <br/>
            2007). Forty-
            <br/>
            eight hours after treatment, the transformed protoplasts are harvested, and
            <br/>
            genomic DNA
            <br/>
            is prepared by a CTAB-based method (Murray and Thompson, Nucl Acids Res, 8:
            <br/>
            4321-
            <br/>
            4325, 1980). Using the genomic DNA prepared from the protoplasts as a
            <br/>
            template, an
            <br/>
            approximately 600-bp fragment encompassing the TALE nuclease recognition site
            <br/>
            is
            <br/>
            10    amplified by PCR. The PCR product is then subjected to 454 pyro-
            <br/>
            sequencing.
            <br/>
            Sequencing reads with insertion/deletion (indel) mutations in the spacer
            <br/>
            region are
            <br/>
            considered as having been derived from imprecise repair of a cleaved TALE
            <br/>
            nuclease
            <br/>
            recognition site by NHEJ. Mutagenesis frequency is calculated as the number of
            <br/>
            <br/>
            sequencing reads with NHEJ mutations out of the total sequencing reads. The
            <br/>
            values are
            <br/>
            15    then normalized by the transformation efficiency. TALE nucleases showing
            <br/>
            activity are
            <br/>
            then used to create lines of soybean with mutations in Gy4 as described below.
            <br/>
            Example 4 ¨ Regeneration of soybean plants containing frameshift mutations
            <br/>
            within Gy4
            <br/>
            Soybean lines with mutations in one or both Gy4 alleles are generated. In
            <br/>
            20    particular, plant parts from soybean (e.g., immature embryos or
            <br/>
            embryogenic callus) are
            <br/>
            bombarded with plasmids encoding TALE nuclease pairs, or transformed via
            <br/>
            Agrobacterium with T-DNA encoding TALE nuclease pairs. Following bombardment,
            <br/>
            plant parts are placed on selection and regeneration media. Materials and
            <br/>
            methods for
            <br/>
            regeneration are used as previously described (Paz et al., Plant Cell Res, 25:
            <br/>
            206-213,
            <br/>
            25    2006). The plasmid and T-DNA contain a selectable marker (e.g.,
            <br/>
            bialaphos) for
            <br/>
            conferring herbicide tolerance and to facilitate selection of transgenic
            <br/>
            plants.
            <br/>
            Transformation efficiencies are monitored using a control plasmid or T-DNA
            <br/>
            plasmid
            <br/>
            containing pNos:YFP and pNos:Bar. To visualize cells or plants that have
            <br/>
            stably
            <br/>
            integrated this control DNA into their genome, a fluorescent stereomicroscope
            <br/>
            is used
            <br/>
            that enables visualization of YFP being expressed in control cells that were
            <br/>
            transformed
            <br/>
            with pNos:YFP and are resistant to bialaphos.
            <br/>
            After delivery of the Gy4-targeted TALE nuclease pair, soybean plants
            <br/>
            containing
            <br/>
            NHEJ mutations are regenerated. Plants containing a deletion of -3(N) ¨ 1
            <br/>
            nucleotides or
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            26
            <br/>
            an insertion of +3(N) + 2 nucleotides, where N is a whole number, including
            <br/>
            zero, are
            <br/>
            advanced to further phenotypic and genome engineering experiments.
            <br/>
            Example 5 ¨ Improving protein stability and folding by restoring the coding
            <br/>
            sequence to the original reading frame
            <br/>
            In some cases, it may be desirable to increase the folding, stability or
            <br/>
            expression
            <br/>
            of the modified protein. For example, the frameshift introduced into Gy4 may
            <br/>
            lead to
            <br/>
            nonsense-mediated decay of the mRNA transcript, thereby reducing Gy4 gene
            <br/>
            expression.
            <br/>
            Further, the modified amino acids within the Gy4 protein may reduce the
            <br/>
            folding
            <br/>
            potential at the C-terminus (folding potential can be calculated using
            <br/>
            publically available
            <br/>
            resources, such as that available at bip.weizmann.acilifldbinifindex, and
            <br/>
            Prilusky et al.,
            <br/>
            Bioinformatics, 21: 3435-3438, 2005).
            <br/>
            One approach to increase the folding, stability and expression of Gy4 is to re-
            <br/>
            <br/>
            introduce the correct reading frame. This is accomplished by designing a
            <br/>
            second pair of
            <br/>
            TALE nucleases that target DNA sequence downstream of the codons of interest,
            <br/>
            but
            <br/>
            upstream of the newly-introduced stop codon (FIG. 5). The desired deletion
            <br/>
            size has a
            <br/>
            total length of -3(N) ¨ 2 where N is a whole number, including zero. The
            <br/>
            desired insertion
            <br/>
            size has a total length of +3(N) + 1 where N is a whole number, including
            <br/>
            zero. In the
            <br/>
            exemplary process, the resulting Gy4 protein, harboring two frameshift
            <br/>
            mutations,
            <br/>
            contains about 3.3% methionine and cysteine, and 7.2% lysine (FIG. 6). A list
            <br/>
            of changes
            <br/>
            to all essential amino acids is provided in TABLE 2.
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072                                           PCT/IB2017/057190
            <br/>
            27
            <br/>
            TABLE 2
            <br/>
            Percent of essential amino acids in Gy4 after introducing a -1 frameshift and
            <br/>
            a second
            <br/>
            frameshift to restore the wild type reading frame.
            <br/>
            Glycine max Gy4 (FIG. 6)
            <br/>
            -1 Frameshift +
            <br/>
            Essential               % amino acid restoration of WT Change from
            <br/>
            Amino Acid in WT protein coding sequence                       WT ( /0)
            <br/>
            ( /0 of amino acid)
            <br/>
            His                          2.78          2.41          -0.38
            <br/>
            Ile                          3.90          3.89          -0.01
            <br/>
            Leu                          6.86          7.96          1.10
            <br/>
            Met                          0.37          2.22          1.85
            <br/>
            Phe                          2.60          2.78          0.18
            <br/>
            Thr                          3.71          5.00          1.29
            <br/>
            Trp                          1.11          1.11          0.00
            <br/>
            Val                          6.49          6.85          0.36
            <br/>
            Lys                          5.01          7.22          2.21
            <br/>
            Example 6 - Circumventing nonsense-mediated decay of mRNA from genes with
            <br/>
            a stop codon before the last intron
            <br/>
            In some cases, it may be desirable to circumvent decreased protein expression
            <br/>
            due
            <br/>
            to nonsense-mediated decay. For example, the frameshift introduced into Gy4 in
            <br/>
            Example
            <br/>
            1 results in a premature stop codon within exon 3. Nonsense-mediate decay is
            <br/>
            avoided by
            <br/>
            c)                                                                 designing a
            <br/>
            second TALE nuclease pair to mutagenize splicing sequences within intron 3,
            <br/>
            thereby preventing processing of the last intron (FIG. 7). Examples of targets
            <br/>
            for
            <br/>
            mutation include, but are not limited to, i) the 5' splice donor site, ii) the
            <br/>
            3' splice
            <br/>
            acceptor site, and iii) the branch site adenosine nucleotide. The resulting
            <br/>
            Gy4 gene,
            <br/>
            harboring two mutations (one frameshift mutation and one intron-inactivating
            <br/>
            mutation)
            <br/>
            produces a Gy4 protein that contains approximately 3.9% methionine and
            <br/>
            cysteine amino
            <br/>
            acids, and 8.8% lysine content. Further, the expression level of the modified
            <br/>
            Gy4 protein
            <br/>
            should be higher than a control that does not contain the intron-inactivating
            <br/>
            mutation
            <br/>
            (FIG. 4).
            <br/>
            Example 7 - Assessing the phenotype of modified soybean plants
            <br/>
            Soybean plants containing frameshift mutations within the Gy4 gene are
            <br/>
            assessed
            <br/>
            for protein composition by two-dimensional protein analysis. Total soluble
            <br/>
            protein is
            <br/>
            isolated from mature seeds as described elsewhere (Schmidt and Herman, Plant
            <br/>
            Biotech
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            28
            <br/>
            J, 6:832-842, 2008). The soluble protein extract (150 mg) from both a modified
            <br/>
            and non-
            <br/>
            modified soybean plant is separated in the first dimension on 11-cm
            <br/>
            immobilized pH
            <br/>
            gradient gel strips (pH 3-10 nonlinear; Bio-Rad) and then in the second
            <br/>
            dimension by
            <br/>
            SDS-PAGE gels (8%-16% linear gradient). The resulting gels are subsequently
            <br/>
            stained
            <br/>
            with 0.1% (w/v) Coomassie Brilliant Blue R250 in 40% (v/v) methanol, 10% (v/v)
            <br/>
            acetic
            <br/>
            acid overnight, and then destained for approximately 3 h in 40% methanol, 10%
            <br/>
            acetic
            <br/>
            acid. The spots on the gels generated from modified plants are compared with
            <br/>
            the spots
            <br/>
            generated from wild type or control plants. Similar intensities in spots that
            <br/>
            represent the
            <br/>
            Gy4 protein between the modified and wild type or control plants suggest that
            <br/>
            the total
            <br/>
            level of methionine and lysine has improved in the modified plants.
            <br/>
            In addition to two-dimensional protein analysis, the overall levels of
            <br/>
            methionine
            <br/>
            and cysteine in the mutant seed are determined by quantitation of hydrolyzed
            <br/>
            amino acids
            <br/>
            and free amino acids using a Waters Acquity ultraperformance liquid
            <br/>
            chromatography
            <br/>
            system (Schmidt, et al., Plant Physiol, 156: 330-345, 2011).
            <br/>
            Example 8 ¨ Increasing lysine and threonine content in wheat by targeting the
            <br/>
            alpha
            <br/>
            gliadins
            <br/>
            Wheat is deficient in the essential amino acids lysine and threonine. To
            <br/>
            increase
            <br/>
            the content of these amino acids in wheat grains, the coding sequence of an
            <br/>
            alpha gliadin
            <br/>
            gene was targeted. A representative alpha gliadin coding sequence from
            <br/>
            Triticum
            <br/>
            aestivum is shown in FIG. 8. The wild type protein contains 2.6% threonine and
            <br/>
            0.7%
            <br/>
            lysine (FIG. 9). To determine if a frameshift can increase the content of
            <br/>
            threonine and
            <br/>
            lysine, the alpha gliadin coding sequence was translated in all three frames
            <br/>
            (FIG. 8).
            <br/>
            Surprisingly, frame -2 contained a very high number of threonine codons and a
            <br/>
            higher
            <br/>
            number of lysine codons, as compared to the wild type sequence. By introducing
            <br/>
            a
            <br/>
            frameshift mutation about 60 bp downstream of the start codon with a deletion
            <br/>
            or
            <br/>
            insertion size of -3(N) - 2 or +3(N) + 1, respectively, the threonine content
            <br/>
            increases from
            <br/>
            2.6% to about 27.8%, and the lysine content increases from 0.7% to 2.3%. There
            <br/>
            are no
            <br/>
            introns within alpha gliadin genes; therefore, it is not necessary to
            <br/>
            introduce a second
            <br/>
            mutation downstream of the frameshift mutation. A list of changes to all
            <br/>
            essential amino
            <br/>
            acids is provided in TABLE 3.
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072                                           PCT/IB2017/057190
            <br/>
            29
            <br/>
            TABLE 3
            <br/>
            Percent of essential amino acids in a representative alpha gliadin protein
            <br/>
            after introducing
            <br/>
            a -2 frameshift near the beginning of the coding sequence
            <br/>
            Triticum aestivum alpha gliadin (FIG. 9)
            <br/>
            Essential                % amino acid -2 Frameshift ( /0 Change from
            <br/>
            Amino Acid in WT protein               of amino acid)    WT ( /0)
            <br/>
            His                           1.68           0.75        -0.93
            <br/>
            Ile                           5.39           8.65        3.26
            <br/>
            Leu                           7.74           4.51        -3.23
            <br/>
            Met                           1.01           1.13        0.12
            <br/>
            Phe                           3.37           1.50        -1.86
            <br/>
            Thr                           2.69           27.82       25.13
            <br/>
            Trp                           0.34           0.38        0.04
            <br/>
            Val                           5.39           4.14        -1.25
            <br/>
            Lys                           0.67           2.26        1.58
            <br/>
            Thus, to generate an alpha gliadin protein with increased threonine content,
            <br/>
            mutations are introduced into an alpha gliadin genomic sequence such that one
            <br/>
            or more
            <br/>
            frameshift mutations of the size -3(N) - 2 or +3(N) + 1 occur, particularly
            <br/>
            within an alpha
            <br/>
            gliadin gene containing the sequence set forth in SEQ ID NO:96
            <br/>
            (ATGAAGACCTTTCTCATCCTTG
            <br/>
            CCCTCCGTGCTATTGTAGCAACCACCGCCACAATT) or within a sequence having
            <br/>
            at least 90% identity to SEQ ID NO:96. In some cases, a TALE nuclease is used
            <br/>
            to
            <br/>
            introduce a frameshift mutation of the size -3(N) - 2 or +3(N) + 1 within the
            <br/>
            alpha gliadin
            <br/>
            sequence, where the mutation is within the sequence set forth in SEQ ID NO:96,
            <br/>
            or
            <br/>
            within a sequence having at least 90% identity to SEQ ID NO:96. The frameshift
            <br/>
            mutation may result in production of an alpha gliadin protein containing the
            <br/>
            amino acid
            <br/>
            sequence set forth in SEQ ID NO:97, or an amino acid sequence having at least
            <br/>
            90%
            <br/>
            identity to SEQ ID NO:97 (TGPGLCPASTTAPV).
            <br/>
            Example 9 - Increasing lysine and threonine content in wheat by targeting
            <br/>
            glutenins
            <br/>
            Increased lysine and threonine content also can be achieved by targeting the
            <br/>
            wheat
            <br/>
            glutenins. A representative high molecular weight glutenin subunit (Glu-1D-1d)
            <br/>
            gene
            <br/>
            from Triticum aestivum is shown in FIG 10A. The wild type protein contains
            <br/>
            2.9%
            <br/>
            threonine and 0.8% lysine (FIG. 11). To determine if a frameshift mutation can
            <br/>
            increase
            <br/>
            the content of threonine amino acids, the glutenin coding sequence was
            <br/>
            translated in all
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            three reading frames (FIG. 10B), revealing that frame -2 contained a very high
            <br/>
            number of
            <br/>
            threonine codons. By introducing a frameshift mutation about 171 bp downstream
            <br/>
            of the
            <br/>
            start codon with a deletion or insertion size of -3(N) - 2 or +3(N) + 1,
            <br/>
            respectively, the
            <br/>
            amino acid content of threonine increases from 3.0% to about 21.4%. A list of
            <br/>
            changes to
            <br/>
            5    all essential amino acids is provided in TABLE 4.
            <br/>
            TABLE 4
            <br/>
            Percent of essential amino acids in a glutenin protein after introducing a -2
            <br/>
            frameshift
            <br/>
            near the beginning of the coding sequence
            <br/>
            Triticum aestivum glutenin (FIG. 11)
            <br/>
            % amino
            <br/>
            Essential                 acid in WT -2 Frameshift ( /0 Change from
            <br/>
            Amino Acid                 protein     of amino acid)    WT ( /0)
            <br/>
            His                           0.47         0.12           -0.35
            <br/>
            Ile                           0.47         2.22           1.75
            <br/>
            Leu                           4.72         4.44           -0.27
            <br/>
            Met                           0.35         1.11           0.76
            <br/>
            Phe                           0.35         3.09           2.73
            <br/>
            Thr                           2.95         21.36          18.41
            <br/>
            Trp                           1.06         0.00           -1.06
            <br/>
            Val                           2.48         4.57           2.09
            <br/>
            Lys                           0.83         0.86           0.04
            <br/>
            10    Thus, to generate a high molecular weight glutenin protein with
            <br/>
            increased
            <br/>
            threonine content, mutations are introduced into a high molecular weight
            <br/>
            glutenin
            <br/>
            genomic sequence such that one or more frameshift mutations of the size -3(N) -
            <br/>
            2 or
            <br/>
            +3(N) + 1 occur, particularly within SEQ ID NO:70 or within a sequence having
            <br/>
            at least
            <br/>
            90% identity to SEQ ID NO:70. In some cases, a TALE nuclease is used to
            <br/>
            introduce a
            <br/>
            15    frameshift mutation of the size -3(N) - 2 or +3(N) + 1 within the a high
            <br/>
            molecular weight
            <br/>
            glutenin sequence, where the mutation is within the sequence set forth in SEQ
            <br/>
            ID NO:70,
            <br/>
            or within a sequence having at least 90% identity to SEQ ID NO:70, where the
            <br/>
            frameshift
            <br/>
            mutation encompasses or follows the nucleotide at position 171 of SEQ ID
            <br/>
            NO:70. The
            <br/>
            frameshift mutation may result in production of a high molecular weight
            <br/>
            glutenin protein
            <br/>
            20    containing the amino acid sequence set forth in SEQ ID NO:98, or an
            <br/>
            amino acid
            <br/>
            sequence having at least 90% identity to SEQ ID NO:98
            <br/>
            (TDRTRAAIRTRATRLLQLIPC).
            <br/>
            The glutenin translation in all three reading frames (FIG. 10B) also showed
            <br/>
            that
            <br/>
            frame -1 contained a high number of lysine amino acids. By introducing a
            <br/>
            frameshift
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072                                           PCT/IB2017/057190
            <br/>
            31
            <br/>
            mutation about 348 bp downstream of the start codon with a deletion or
            <br/>
            insertion size
            <br/>
            of -3(N) - 1 or +3(N) + 2, respectively, the amino acid content of lysine
            <br/>
            increases from
            <br/>
            0.8% to about 8.7%. A list of changes to all essential amino acids is provided
            <br/>
            in TABLE
            <br/>
            5. Surprisingly, all essential amino acids, with the exception of tryptophan,
            <br/>
            increase in
            <br/>
            content within the protein produced from this -1 frameshift.
            <br/>
            TABLE 5
            <br/>
            Percent of essential amino acids in a glutenin protein after introducing a -1
            <br/>
            frameshift
            <br/>
            near the beginning of the glutenin coding sequence
            <br/>
            Triticum aestivum glutenin (FIG. 11)
            <br/>
            -2 Frameshift
            <br/>
            Essential               % amino acid in    ( /0 of amino    Change from
            <br/>
            Amino Acid               WT protein        acid)        WT ( /0)
            <br/>
            His                          0.47          1.35          0.88
            <br/>
            Ile                          0.47          1.69          1.22
            <br/>
            Leu                          4.72          8.45          3.73
            <br/>
            Met                          0.35          0.68          0.32
            <br/>
            Phe                          0.35          1.35          1.00
            <br/>
            Thr                          2.95          4.73          1.78
            <br/>
            Trp                          1.06          0.34          -0.72
            <br/>
            Val                          2.48          7.09          4.62
            <br/>
            Lys                          0.83          8.78          7.96
            <br/>
            Thus, to generate a high molecular weight glutenin protein with increased
            <br/>
            lysine
            <br/>
            content, mutations are introduced into a high molecular weight glutenin
            <br/>
            genomic
            <br/>
            sequence such that one or more frameshift mutations of the size -3(N) - 1 or
            <br/>
            +3(N) + 2
            <br/>
            occur, particularly within SEQ ID NO:70 or within a sequence having at least
            <br/>
            90%
            <br/>
            identity to SEQ ID NO:70. In some cases, a TALE nuclease is used to introduce
            <br/>
            a
            <br/>
            frameshift mutation of the size -3(N) - 1 or +3(N) + 2 within the a high
            <br/>
            molecular weight
            <br/>
            glutenin sequence, where the mutation is within the sequence set forth in SEQ
            <br/>
            ID NO:70,
            <br/>
            or within a sequence having at least 90% identity to SEQ ID NO:70, where the
            <br/>
            frameshift
            <br/>
            mutation encompasses or follows the nucleotide at position 348 of SEQ ID
            <br/>
            NO:70. The
            <br/>
            frameshift mutation may result in production of a high molecular weight
            <br/>
            glutenin protein
            <br/>
            containing the amino acid sequence set forth in SEQ ID NO:99, or an amino acid
            <br/>
            sequence having at least 90% identity to SEQ ID NO:99
            <br/>
            (LLCNSRDKGNQGTTQLLCSS).
            <br/>
            <br/>
            CA 03042857 2019-05-03
            <br/>
            WO 2018/092072
            <br/>
            PCT/IB2017/057190
            <br/>
            32
            <br/>
            OTHER EMBODIMENTS
            <br/>
            It is to be understood that while the invention has been described in
            <br/>
            conjunction
            <br/>
            with the detailed description thereof, the foregoing description is intended
            <br/>
            to illustrate and
            <br/>
            not limit the scope of the invention, which is defined by the scope of the
            <br/>
            appended claims.
            <br/>
            Other aspects, advantages, and modifications are within the scope of the
            <br/>
            following
            <br/>
            claims.
            <br/>
            <br/>
           </div>
          </div>
         </div>
        </details>
        <!-- Representative Drawing -->
        <details class="acc-group" id="tabs1_4">
         <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group", "persist": "session"}'>
          <span class="h6">
           Representative Drawing
          </span>
         </summary>
         <div class="tgl-panel">
          <div class="panel-body">
           <img alt="A single figure which represents the drawing illustrating the invention." src="/opic-cipo/cpd/page/3042857_20190016_page1_scale25_rotate0_objectnameA1001001A19E16C21633B08832.gif"/>
          </div>
          <div class="clearfix">
          </div>
         </div>
        </details>
        <!-- Admin Status -->
        <details class="acc-group" id="tabs1_5">
         <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group", "persist": "session"}'>
          <span class="h6">
           Administrative Status
          </span>
         </summary>
         <div class="tgl-panel">
          <div class="alert alert-info">
           <p>
            <strong>
             For a clearer understanding of the status of the application/patent presented on this page, the site
             <a aria-label="DisclaimerThis link opens the help page." href="/opic-cipo/cpd/eng/disclaimer.html" id="aDisclaimer" onclick="saveFocusId();" title="Canadian Patents Database usage disclaimer">
              Disclaimer
             </a>
             , as well as the definitions for
             <a aria-label="PatentThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_general_information.html#patent" id="aHelpGeneralInfo" onclick="saveFocusId();" title="A patent is a government grant giving the exclusive right of making, using or selling an invention.">
              Patent
             </a>
             ,
             <a aria-label="Administrative StatusThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_administrative_status.html#admin" id="aHelpAdminStatus" onclick="saveFocusId();" title="Help on administrative status fields and dates associated with the processing of patent applications">
              Administrative Status
             </a>
             ,
             <a aria-label="Maintenance FeeThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_maintenance_fees.html#maintfee" id="aHelpMaintenanceFee" onclick="saveFocusId();" title="Help on maintenance fee information associated with patents">
              Maintenance Fee
             </a>
             and
             <a aria-label="Payment HistoryThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_payment_history.html#Hist" id="aHelpPaymentHistory" onclick="saveFocusId();" title="Help on payment amounts and date fields associated with patents">
              Payment History
             </a>
             should be consulted.
            </strong>
           </p>
          </div>
          <!-- Current Status -->
          <!-- To be re-enabled as part of JIRA CIPOPWEB-1019  -->
          <!-- 
					<div class="section">
						<h2>Current Status</h2>
						<table class="table table-bordered col-lg-12" id="currentStatusTable" title="Table showing current status information">
							<thead>
								<tr class="text-center">
									<th class="col-lg-12 active" scope="col">Compliant</th>
								</tr>
							</thead>
						</table>
						<div class="clearfix"></div>
					</div>
					 -->
          <!-- Administrative Status -->
          <div class="section">
           <h2>
            Administrative Status
           </h2>
           <table class="table table-bordered col-lg-8" id="administrativeStatusTable" title="Table showing administrative status information">
            <thead>
             <tr class="text-center">
              <th class="col-lg-4 active" scope="col">
               Title
              </th>
              <th class="col-lg-3 active" scope="col">
               Date
              </th>
             </tr>
            </thead>
            <tbody>
             <tr>
              <td class="text-left">
               <a aria-label="Forecasted Issue DateThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_administrative_status.html#grondt" id="0" onclick="return saveFocusId();" title="Date in which CIPO expects to issue the patent. This is subject to change without notice">
                Forecasted Issue Date
               </a>
              </td>
              <td class="text-left">
               Unavailable
              </td>
             </tr>
             <tr>
              <td class="text-left">
               <a aria-label="(86) PCT Filing DateThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_administrative_status.html#AAFI" id="AAFI" onclick="return saveFocusId();" title="The date  the PCT application was filed with WIPO">
                (86) PCT Filing Date
               </a>
              </td>
              <td class="text-left nowrap">
               2017-11-16
              </td>
             </tr>
             <tr>
              <td class="text-left">
               <a aria-label="(87) PCT Publication DateThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_administrative_status.html#BBLO" id="BBLO" onclick="return saveFocusId();" title="The date the PCT application was made available for public inspection (viewing by the public)">
                (87) PCT Publication Date
               </a>
              </td>
              <td class="text-left nowrap">
               2018-05-24
              </td>
             </tr>
             <tr>
              <td class="text-left">
               <a aria-label="(85) National EntryThis link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_administrative_status.html#AFNE" id="AFNE" onclick="return saveFocusId();" title="The date the PCT application was given a Canadian filing date (entered the Canadian phase)">
                (85) National Entry
               </a>
              </td>
              <td class="text-left nowrap">
               2019-05-03
              </td>
             </tr>
            </tbody>
           </table>
           <div class="clearfix">
           </div>
          </div>
          <!-- Abandonment status. -->
          <div class="section">
           <h2>
            Abandonment History
           </h2>
           <p>
            There is no abandonment history.
           </p>
          </div>
          <!--  Court order information. -->
          <div class="clearfix">
          </div>
          <!-- Maintenance Fee -->
          <div class="section">
           <h3>
            Maintenance Fee
           </h3>
           <p>
            <a href="/opic-cipo/cpd/eng/help/content/help_maintenance_fees.html#LA" title="Most recent maintenace fee paid.  If the fees are pre-paid, only the most recent will be shown">
             Last Payment
            </a>
            of
                    $100.00
                    was received on
                    2021-10-11
           </p>
           <p>
            <br/>
            <b>
             Upcoming maintenance fee amounts
            </b>
           </p>
           <table class="table table-bordered col-lg-8" id="maintenanceFeeTable" title="Table showing maintenance fee information">
            <thead>
             <tr class="text-center">
              <th class="col-lg-6 active" scope="col">
               Description
              </th>
              <th class="col-lg-1 active" scope="col">
               Date
              </th>
              <th class="col-lg-1 active" scope="col">
               Amount
              </th>
             </tr>
            </thead>
            <tbody>
             <tr>
              <td class="alignLeft">
               <a href="/opic-cipo/cpd/eng/help/content/help_maintenance_fees.html#NX" id="aLargeEntity" onclick="saveFocusId();" title="Date the next entity fee is due ">
                Next Payment
               </a>
               if standard fee
              </td>
              <td class="alignCenter nowrap">
               2022-11-16
              </td>
              <td class="alignRight nowrap">
               $203.59
              </td>
             </tr>
             <tr>
              <td class="alignLeft">
               <a href="/opic-cipo/cpd/eng/help/content/help_maintenance_fees.html#NX" id="aSmallEntity" onclick="saveFocusId();" title="Date the next entity fee is due ">
                Next Payment
               </a>
               if small entity fee
              </td>
              <td class="alignCenter nowrap">
               2022-11-16
              </td>
              <td class="alignRight nowrap">
               $100.00
              </td>
             </tr>
            </tbody>
           </table>
           <div class="clear">
           </div>
           <p>
            <strong>
             Note
            </strong>
            : If the full payment has not been received on or before the date indicated, a further fee may be required which may be one of the following
           </p>
           <ul>
            <li>
             the reinstatement fee;
            </li>
            <li>
             the late payment fee; or
            </li>
            <li>
             additional fee to reverse deemed expiry.
            </li>
           </ul>
           <p>
            Patent fees are adjusted on the 1st of January every year. The amounts above are the current amounts if received by December 31 of the current year.
            <br/>
            Please refer to the CIPO
            <a href="https://cipo.ic.gc.ca/eic/site/cipointernet-internetopic.nsf/eng/wr00142.html" id="aPatentFee" onclick="saveFocusId();" title="Patent Fees">
             Patent Fees
            </a>
            web page to see all current fee amounts.
           </p>
          </div>
          <!-- Payment History -->
          <div class="section">
           <h2>
            Payment History
           </h2>
           <table class="table table-bordered col-lg-8" id="paymentHistoryTable" title="Table showing payment history information">
            <thead>
             <tr class="text-center">
              <th class="col-lg-3 active" scope="col">
               Fee Type
              </th>
              <th class="col-lg-1 active" scope="col">
               Anniversary Year
              </th>
              <th class="col-lg-1 active" scope="col">
               Due Date
              </th>
              <th class="col-lg-1 active" scope="col">
               Amount Paid
              </th>
              <th class="col-lg-1 active" scope="col">
               Paid Date
              </th>
             </tr>
            </thead>
            <tbody>
             <tr>
              <td>
               <a href="/opic-cipo/cpd/eng/help/content/help_payment_history.html#REG" id="Registration-of-a-document---section-1241" onclick="saveFocusId();" title="The most recent date and fee paid to register a document relating to a patent or application">
                Registration of a document - section 124
               </a>
              </td>
              <td>
              </td>
              <td>
              </td>
              <td class="text-right nowrap">
               $100.00
              </td>
              <td class="text-center nowrap">
               2019-05-03
              </td>
             </tr>
             <tr>
              <td>
               <a href="/opic-cipo/cpd/eng/help/content/help_payment_history.html#REG" id="Registration-of-a-document---section-1242" onclick="saveFocusId();" title="The most recent date and fee paid to register a document relating to a patent or application">
                Registration of a document - section 124
               </a>
              </td>
              <td>
              </td>
              <td>
              </td>
              <td class="text-right nowrap">
               $100.00
              </td>
              <td class="text-center nowrap">
               2019-05-03
              </td>
             </tr>
             <tr>
              <td>
               <a href="/opic-cipo/cpd/eng/help/content/help_payment_history.html#FIL" id="Application-Fee3" onclick="saveFocusId();" title="Date the application filing fee was paid">
                Application Fee
               </a>
              </td>
              <td>
              </td>
              <td>
              </td>
              <td class="text-right nowrap">
               $400.00
              </td>
              <td class="text-center nowrap">
               2019-05-03
              </td>
             </tr>
             <tr>
              <td>
               <a href="/opic-cipo/cpd/eng/help/content/help_payment_history.html#MAN" id="Maintenance-Fee---Application---New-Act4" onclick="saveFocusId();" title="Maintenance fee information for an application filed on or after October 1, 1989">
                Maintenance Fee - Application - New Act
               </a>
              </td>
              <td class="text-center">
               2
              </td>
              <td class="text-center nowrap">
               2019-11-18
              </td>
              <td class="text-right nowrap">
               $100.00
              </td>
              <td class="text-center nowrap">
               2019-10-29
              </td>
             </tr>
             <tr>
              <td>
               <a href="/opic-cipo/cpd/eng/help/content/help_payment_history.html#MAN" id="Maintenance-Fee---Application---New-Act5" onclick="saveFocusId();" title="Maintenance fee information for an application filed on or after October 1, 1989">
                Maintenance Fee - Application - New Act
               </a>
              </td>
              <td class="text-center">
               3
              </td>
              <td class="text-center nowrap">
               2020-11-16
              </td>
              <td class="text-right nowrap">
               $100.00
              </td>
              <td class="text-center nowrap">
               2020-11-06
              </td>
             </tr>
             <tr>
              <td>
               <a href="/opic-cipo/cpd/eng/help/content/help_payment_history.html#MAN" id="Maintenance-Fee---Application---New-Act6" onclick="saveFocusId();" title="Maintenance fee information for an application filed on or after October 1, 1989">
                Maintenance Fee - Application - New Act
               </a>
              </td>
              <td class="text-center">
               4
              </td>
              <td class="text-center nowrap">
               2021-11-16
              </td>
              <td class="text-right nowrap">
               $100.00
              </td>
              <td class="text-center nowrap">
               2021-10-11
              </td>
             </tr>
            </tbody>
           </table>
          </div>
          <div class="clearfix">
          </div>
         </div>
        </details>
        <!-- Owners on Record -->
        <details class="acc-group" id="tabs1_6">
         <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group", "persist": "session"}'>
          <span class="h6">
           Owners on Record
          </span>
         </summary>
         <div class="tgl-panel">
          <div class="section">
           <div class="alert alert-info">
            <p>
             Note: Records showing the ownership history in alphabetical order.
            </p>
           </div>
           <table class="table table-bordered col-lg-5" id="currentOwnersTable" title="Table showing the ownership history">
            <thead>
             <tr class="text-center">
              <th class="active" scope="col">
               Current Owners on Record
              </th>
             </tr>
            </thead>
            <tbody>
             <tr>
              <td class="text-left">
               CELLECTIS
              </td>
             </tr>
            </tbody>
           </table>
           <div class="clearfix">
           </div>
          </div>
          <div class="section">
           <table class="table table-bordered col-lg-8" id="ownershipHistoryTable" title="Table showing the ownership history">
            <thead>
             <tr class="text-center">
              <th class="active" scope="col">
               Past Owners on Record
              </th>
             </tr>
            </thead>
            <tbody>
             <tr>
              <td class="text-left">
               None
              </td>
             </tr>
            </tbody>
           </table>
           <div class="clearfix">
           </div>
           <div class="mrgn-left-lg">
            Past Owners that do not appear in the "Owners on Record" listing will appear in other documentation within the application.
           </div>
           <div class="clearfix">
           </div>
          </div>
         </div>
        </details>
        <!-- Documents -->
        <details class="acc-group" id="tabs1_7">
         <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group", "persist": "session"}'>
          <span class="h6">
           Documents
          </span>
         </summary>
         <div class="tgl-panel">
          <form action="/opic-cipo/cpd/eng/patent/3042857/summary.html" id="documentForm" method="post" name="documentForm">
           <input id="downloadZipFlag" name="downloadZipFlag" type="hidden" value="N"/>
           <input id="downloadCombinedFlag" name="downloadCombinedFlag" type="hidden" value="N"/>
           <input id="captchaInvalid" name="captchaInvalid" type="hidden" value="N"/>
           <input id="type" name="type" type="hidden" value="number_search"/>
           <input id="start" name="start" type="hidden" value=""/>
           <input id="num" name="num" type="hidden" value=""/>
           <input id="query" name="query" type="hidden" value=""/>
           <input id="downloadTokenValue" name="downloadTokenValue" type="hidden"/>
           <input id="fileDownloadToken" name="fileDownloadToken" type="hidden"/>
           <div class="alert alert-danger" id="catpchaInvalid">
           </div>
           <div id="catpcha">
            <br/>
            <p class="h4">
             <strong>
              To view selected files, please enter reCAPTCHA code :
             </strong>
            </p>
            <div class="g-recaptcha" data-sitekey="6LcTxQATAAAAAHfRmJId2aDpF8XyEfAxCouBSozi">
            </div>
            <br/>
            <ul class="list-unstyled list-inline">
             <li>
              <input class="btn btn-primary" type="submit" value="Download documents"/>
             </li>
             <li>
              <input onclick="cancelDownload()" type="button" value="Cancel"/>
             </li>
            </ul>
            <div class="clearfix">
            </div>
            <br/>
           </div>
           <script type="text/javascript">
            $('#catpcha').hide();
						  		$('#catpchaInvalid').hide();
           </script>
           <div id="mainSection">
            <div class="alert alert-info">
             <p>
              To view images, click a link in the Document Description column. To download the documents, select one or more checkboxes in the first column and then click the "Download Selected in PDF format (Zip Archive)" button.
             </p>
             <p>
              <a href="https://cipo.ic.gc.ca/eic/site/cipointernet-internetopic.nsf/eng/wr03929.html" id="aPublishedDoc" onclick="saveFocusId();" title="View the List of published and non-published patent-specific documents on the CPD">
               List of published and non-published patent-specific documents on the CPD
              </a>
              .
             </p>
             <p>
              If you have any difficulty accessing content,  you can call the Client Service Centre at 1-866-997-1936 or send them an e-mail at
              <a href="mailto:cipo.contact@ic.gc.ca" title="E-mail the Canadian Intellectual Property Office">
               CIPO Client Service Centre
              </a>
              .
             </p>
            </div>
            <div class="clearfix">
            </div>
            <div class="mrgn-tp-md">
             <div class="col-lg-5">
              <label class="control-label" for="displayGroupA">
               Filter
              </label>
              <select class="form-control" id="displayGroupA" name="displayGroup" onchange="submitForm(displayGroupA.value)">
               <option selected="" value="010">
                All Documents
               </option>
               <option value="020">
                As Filed
               </option>
               <option value="030">
                As Open to Public Inspection
               </option>
               <option value="040">
                At Issuance
               </option>
               <option value="050">
                Examination
               </option>
               <option value="060">
                Incoming Correspondence
               </option>
               <option value="070">
                Miscellaneous
               </option>
               <option value="080">
                Outgoing Correspondence
               </option>
               <option value="090">
                Payment
               </option>
              </select>
             </div>
             <div class="col-lg-7 mrgn-tp-sm">
              <!-- <div class="col-lg-7 mrgn-tp-sm"> -->
              <br/>
              <a class="btn btn-primary small" href="javascript:void(0)" id="zipLink" onclick="showRecaptcha()" title="Download Selected in PDF format (Zip Archive)">
               <span class="glyphicon glyphicon-download">
               </span>
               Download Selected in PDF format (Zip Archive)
              </a>
              <!-- </div> -->
             </div>
             <div class="clearfix">
             </div>
             <hr/>
             <!--                          <div class="table-responsive container-fluid row small"> -->
             <table aria-labelledby="phGridLabel" class="table table-hover table-bordered" data-wb-tables='{ "paging": false, "lengthMenu": [[ 10, 25, 50, 100, -1], [ 10, 25, 50, 100, "All"]] ,"columnDefs": [  { "orderable": true } ], "order": [2 , "desc"] }' id="documentsTable" role="grid" title="Table listing patent images for viewing or downloading">
              <thead>
               <tr class="text-center">
                <th class="active" scope="col">
                 Select
                </th>
                <th class="active" scope="col">
                 Document
                 <br/>
                 Description
                 <span class="sorting-icons">
                 </span>
                </th>
                <th class="active" scope="col">
                 Date
                 <br/>
                 (yyyy-mm-dd)
                 <span class="sorting-icons">
                 </span>
                </th>
                <th class="active" scope="col">
                 Number of pages
                 <span class="sorting-icons">
                 </span>
                </th>
                <th class="active" scope="col">
                 Size of Image (KB)
                 <span class="sorting-icons">
                 </span>
                </th>
               </tr>
              </thead>
              <tbody>
               <tr>
                <td>
                 <label for="documentCheckedId0">
                 </label>
                 <input id="documentCheckedId0" name="documentChecked" type="checkbox" value="A1001001A19E15B00123G94748"/>
                </td>
                <td class="text-left">
                 <!--  Variable used to know if a slash ("/") will be needed between doc types -->
                 <!-- Document Type needs to be lower case and trimmed -->
                 <a href="/opic-cipo/cpd/eng/patent/3042857/images.html?modificationDate=20190003&amp;page=1&amp;scale=25&amp;rotation=0&amp;frenchDocType=Abr%c3%a9g%c3%a9&amp;englishDocType=Abstract&amp;type=number_search&amp;objectName=A1001001A19E15B00123G94748&amp;numPages=2" id="documentLink0URL0" onclick="saveFocusId();" title="Click to view image(HTML) of  Abstract">
                  Abstract
                 </a>
                 <!-- Add a slash "/" if there is more than one document -->
                </td>
                <td class="text-center">
                 2019-05-03
                </td>
                <td class="text-right">
                 2
                </td>
                <td class="text-right">
                 67
                </td>
               </tr>
               <tr>
                <td>
                 <label for="documentCheckedId1">
                 </label>
                 <input id="documentCheckedId1" name="documentChecked" type="checkbox" value="A1001001A19E15B00124B94752"/>
                </td>
                <td class="text-left">
                 <!--  Variable used to know if a slash ("/") will be needed between doc types -->
                 <!-- Document Type needs to be lower case and trimmed -->
                 <a href="/opic-cipo/cpd/eng/patent/3042857/images.html?modificationDate=20190003&amp;page=1&amp;scale=25&amp;rotation=0&amp;frenchDocType=Revendications&amp;englishDocType=Claims&amp;type=number_search&amp;objectName=A1001001A19E15B00124B94752&amp;numPages=8" id="documentLink1URL0" onclick="saveFocusId();" title="Click to view image(HTML) of  Claims">
                  Claims
                 </a>
                 <!-- Add a slash "/" if there is more than one document -->
                </td>
                <td class="text-center">
                 2019-05-03
                </td>
                <td class="text-right">
                 8
                </td>
                <td class="text-right">
                 272
                </td>
               </tr>
               <tr>
                <td>
                 <label for="documentCheckedId2">
                 </label>
                 <input id="documentCheckedId2" name="documentChecked" type="checkbox" value="A1001001A19E15B00124J94756"/>
                </td>
                <td class="text-left">
                 <!--  Variable used to know if a slash ("/") will be needed between doc types -->
                 <!-- Document Type needs to be lower case and trimmed -->
                 <a href="/opic-cipo/cpd/eng/patent/3042857/images.html?modificationDate=20190003&amp;page=1&amp;scale=25&amp;rotation=0&amp;frenchDocType=Dessins&amp;englishDocType=Drawings&amp;type=number_search&amp;objectName=A1001001A19E15B00124J94756&amp;numPages=13" id="documentLink2URL0" onclick="saveFocusId();" title="Click to view image(HTML) of  Drawings">
                  Drawings
                 </a>
                 <!-- Add a slash "/" if there is more than one document -->
                </td>
                <td class="text-center">
                 2019-05-03
                </td>
                <td class="text-right">
                 13
                </td>
                <td class="text-right">
                 585
                </td>
               </tr>
               <tr>
                <td>
                 <label for="documentCheckedId3">
                 </label>
                 <input id="documentCheckedId3" name="documentChecked" type="checkbox" value="A1001001A19E15B00125E94762"/>
                </td>
                <td class="text-left">
                 <!--  Variable used to know if a slash ("/") will be needed between doc types -->
                 <!-- Document Type needs to be lower case and trimmed -->
                 <a href="/opic-cipo/cpd/eng/patent/3042857/images.html?modificationDate=20190003&amp;page=1&amp;scale=25&amp;rotation=0&amp;frenchDocType=Description&amp;englishDocType=Description&amp;type=number_search&amp;objectName=A1001001A19E15B00125E94762&amp;numPages=32" id="documentLink3URL0" onclick="saveFocusId();" title="Click to view image(HTML) of  Description">
                  Description
                 </a>
                 <!-- Add a slash "/" if there is more than one document -->
                </td>
                <td class="text-center">
                 2019-05-03
                </td>
                <td class="text-right">
                 32
                </td>
                <td class="text-right">
                 1,636
                </td>
               </tr>
               <tr>
                <td>
                 <label for="documentCheckedId4">
                 </label>
                 <input id="documentCheckedId4" name="documentChecked" type="checkbox" value="A1001001A19E15B00125J94766"/>
                </td>
                <td class="text-left">
                 <!--  Variable used to know if a slash ("/") will be needed between doc types -->
                 <!-- Document Type needs to be lower case and trimmed -->
                 <a href="/opic-cipo/cpd/eng/patent/3042857/images.html?modificationDate=20190003&amp;page=1&amp;scale=25&amp;rotation=0&amp;frenchDocType=Dessins+repr%c3%a9sentatifs&amp;englishDocType=Representative+Drawing&amp;type=number_search&amp;objectName=A1001001A19E15B00125J94766&amp;numPages=1" id="documentLink4URL0" onclick="saveFocusId();" title="Click to view image(HTML) of  Representative Drawing">
                  Representative Drawing
                 </a>
                 <!-- Add a slash "/" if there is more than one document -->
                </td>
                <td class="text-center">
                 2019-05-03
                </td>
                <td class="text-right">
                 1
                </td>
                <td class="text-right">
                 19
                </td>
               </tr>
               <tr>
                <td>
                 <label for="documentCheckedId5">
                 </label>
                 <input id="documentCheckedId5" name="documentChecked" type="checkbox" value="A1001001A19E15B00126D94768"/>
                </td>
                <td class="text-left">
                 <!--  Variable used to know if a slash ("/") will be needed between doc types -->
                 <!-- Document Type needs to be lower case and trimmed -->
                 <a href="/opic-cipo/cpd/eng/patent/3042857/images.html?modificationDate=20190003&amp;page=1&amp;scale=25&amp;rotation=0&amp;frenchDocType=Rapport+de+recherche+internationale&amp;englishDocType=International+Search+Report&amp;type=number_search&amp;objectName=A1001001A19E15B00126D94768&amp;numPages=9" id="documentLink5URL0" onclick="saveFocusId();" title="Click to view image(HTML) of  International Search Report">
                  International Search Report
                 </a>
                 <!-- Add a slash "/" if there is more than one document -->
                </td>
                <td class="text-center">
                 2019-05-03
                </td>
                <td class="text-right">
                 9
                </td>
                <td class="text-right">
                 257
                </td>
               </tr>
               <tr>
                <td>
                 <label for="documentCheckedId6">
                 </label>
                 <input id="documentCheckedId6" name="documentChecked" type="checkbox" value="A1001001A19E15B00126H94772"/>
                </td>
                <td class="text-left">
                 <!--  Variable used to know if a slash ("/") will be needed between doc types -->
                 <!-- Document Type needs to be lower case and trimmed -->
                 <a href="/opic-cipo/cpd/eng/patent/3042857/images.html?modificationDate=20190003&amp;page=1&amp;scale=25&amp;rotation=0&amp;frenchDocType=D%c3%a9claration&amp;englishDocType=Declaration&amp;type=number_search&amp;objectName=A1001001A19E15B00126H94772&amp;numPages=3" id="documentLink6URL0" onclick="saveFocusId();" title="Click to view image(HTML) of  Declaration">
                  Declaration
                 </a>
                 <!-- Add a slash "/" if there is more than one document -->
                </td>
                <td class="text-center">
                 2019-05-03
                </td>
                <td class="text-right">
                 3
                </td>
                <td class="text-right">
                 49
                </td>
               </tr>
               <tr>
                <td>
                 <label for="documentCheckedId7">
                 </label>
                 <input id="documentCheckedId7" name="documentChecked" type="checkbox" value="A1001001A19E15B00127B94776"/>
                </td>
                <td class="text-left">
                 <!--  Variable used to know if a slash ("/") will be needed between doc types -->
                 <!-- Document Type needs to be lower case and trimmed -->
                 <a href="/opic-cipo/cpd/eng/patent/3042857/images.html?modificationDate=20190003&amp;page=1&amp;scale=25&amp;rotation=0&amp;frenchDocType=Demande+d'entr%c3%a9e+en+phase+nationale&amp;englishDocType=National+Entry+Request&amp;type=number_search&amp;objectName=A1001001A19E15B00127B94776&amp;numPages=12" id="documentLink7URL0" onclick="saveFocusId();" title="Click to view image(HTML) of  National Entry Request">
                  National Entry Request
                 </a>
                 <!-- Add a slash "/" if there is more than one document -->
                </td>
                <td class="text-center">
                 2019-05-03
                </td>
                <td class="text-right">
                 12
                </td>
                <td class="text-right">
                 348
                </td>
               </tr>
               <tr>
                <td>
                 <label for="documentCheckedId8">
                 </label>
                 <input id="documentCheckedId8" name="documentChecked" type="checkbox" value="A1001001A19E28B60059H83779"/>
                </td>
                <td class="text-left">
                 <!--  Variable used to know if a slash ("/") will be needed between doc types -->
                 <!-- Document Type needs to be lower case and trimmed -->
                 <a href="/opic-cipo/cpd/eng/patent/3042857/images.html?modificationDate=20190028&amp;page=1&amp;scale=25&amp;rotation=0&amp;frenchDocType=Page+couverture&amp;englishDocType=Cover+Page&amp;type=number_search&amp;objectName=A1001001A19E28B60059H83779&amp;numPages=1" id="documentLink8URL0" onclick="saveFocusId();" title="Click to view image(HTML) of  Cover Page">
                  Cover Page
                 </a>
                 <!-- Add a slash "/" if there is more than one document -->
                </td>
                <td class="text-center">
                 2019-05-28
                </td>
                <td class="text-right">
                 1
                </td>
                <td class="text-right">
                 40
                </td>
               </tr>
              </tbody>
             </table>
             <!-- </div> -->
             <script async="" defer="" src="https://www.google.com/recaptcha/api.js?hl=en&amp;onload=CaptchaCallback&amp;render=explicit">
             </script>
             <script type="text/javascript">
              var CaptchaCallback = function() {
						  $('.g-recaptcha').each(function(index, el) {
						    grecaptcha.render(el, {'sitekey' : '6LcTxQATAAAAAHfRmJId2aDpF8XyEfAxCouBSozi'});
						  });
						};
             </script>
             <br/>
             <script src="/opic-cipo/cpd/view/scripts/jquery.blockUI.js" type="text/javascript">
             </script>
             <script src="/opic-cipo/cpd/view/scripts/jquery.cookie.js" type="text/javascript">
             </script>
             <div aria-live="assertive" id="domMessage" style="display: none;">
              <!-- This does not work in Internet Explorer -->
              <h2>
               Your request is in progress.
               <br/>
               <br/>
               Requested information will be available
               <br/>
               in a moment.
               <br/>
               <br/>
               Thank you for waiting.
              </h2>
              <img alt="Request in progress image " class="image-actual text-center" src="/opic-cipo/cpd/images/progress.gif"/>
             </div>
             <script>
              function cancelDownload() {  
							//Hide capatcha section and show Main Section		
							$('#catpchaInvalid').hide();	
							$('#catpcha').hide();
							$('#mainSection').show();
							document.getElementById("mainSection").setAttribute("role", "alert");	
						}
						
						function showRecaptcha() {  		
							if ( $('input[name="documentChecked"]').is(':checked') ) { 
								if ($('#catpcha').not(':visible')){							
									$('#catpcha').show();						
									document.getElementById("catpcha").setAttribute("role", "alert");			
									$('#mainSection').hide();
								}		
							}
						}
						
						function showRecaptcha2() {  		
							if ($('#catpcha').not(':visible')){							
								$('#catpcha').show();						
								document.getElementById("catpcha").setAttribute("role", "alert");			
								$('#mainSection').hide();
							}		
						}
			
						
						$(document).ready(function () {
						    $('#documentForm').submit(function () {      
						      blockUIForDownload();
						    });
						    
						  });
						
						function preDownloadCombined() {
						  document.getElementById("downloadCombinedFlag").value = "Y";
						}
						
						function preDownloadZip() {
						  document.getElementById("downloadZipFlag").value = "Y";
						}
						
						  var fileDownloadCheckTimer;
						  function blockUIForDownload() {
						    //use the current timestamp as the token value
						    var token = new Date().getTime();    
						    $('#downloadTokenValue').val(token);
						    $('#downloadZipFlag').val("Y");
							document.getElementById("domMessage").setAttribute("role", "alert");	
							
							if(navigator.userAgent.indexOf("MSIE") != -1 ) {
			    				var pb = document.getElementById("domMessage");
			    				pb.innerHTML = '<h2>Your request is in progress.<br><br>Requested information will be available<br>in a moment.<br><br>Thank you for waiting.</h2><img src="/opic-cipo/cpd/images/progress.gif" class="image-actual text-center" alt="Request in progress image " />';    				    				    				    			
							}
							
						    $.blockUI({ 
						    	message: $('#domMessage'),
						    	css: { backgroundColor: '#FFFFFF', color: '#000000', border: '3px solid #6487C6' } 
						    	}); 
						    	
						    fileDownloadCheckTimer = window.setInterval(function () {	  
							  var cookieValue = $.cookie("fileDownloadToken");
						      if (cookieValue == token)
						       finishDownload();
						    }, 1000);
						  }
						  
									
						function submitForm(groupSelection) {
						  	if (document.getElementById("displayGroupA")) { 
						  		document.getElementById("displayGroupA").value = groupSelection;  		
						  	}
							document.getElementById("downloadZipFlag").value = "N";
							document.getElementById("downloadCombinedFlag").value = "N";
							document.getElementById("tabs1Index").value = "tabs1_7";			
							document.documentForm.submit();
							document.querySelector("#tabs1_7").scrollIntoView({ behavior: 'smooth', block: 'start'});
						}
						
						function finishDownload() {
							 window.clearInterval(fileDownloadCheckTimer);
							 $.removeCookie('fileDownloadToken'); //clears this cookie value
							 $.unblockUI(); 
							 $('#catpcha').hide();
							 $('#catpchaInvalid').hide();					
							 $('#mainSection').show();
							 grecaptcha.reset();
							//<!-- The following is only for Internet Explorer -->
							 if(navigator.userAgent.indexOf("MSIE") != -1 ) {
								$('#domMessage').hide();
							 }
						}
						function scrollAfterFilter() {
							document.querySelector("#tabs1_7").scrollIntoView({ behavior: 'smooth', block: 'start'});
						}
             </script>
            </div>
            <!--  Download Biological Sequence Listings -->
            <details class="acc-group" id="tabs1_8">
             <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group", "persist": "session"}'>
              <span class="h6">
               Biological Sequence Listings
              </span>
             </summary>
             <div class="tgl-panel">
              <div class="section">
               <div class="alert alert-info">
                <p>
                 Choose a
                 <abbr title="Biological Sequence Listings">
                  BSL
                 </abbr>
                 submission then click the "Download BSL" button to
                 <a aria-label="This link opens the help page." href="/opic-cipo/cpd/eng/help/content/help_document_images.html#download_BSL" title="Help on downloading Biological Sequence Listings">
                  download
                 </a>
                 the file.
                </p>
                <p>
                 If you have any difficulty accessing content,  you can call the Client Service Centre at 1-866-997-1936 or send them an e-mail at
                 <a href="mailto:cipo.contact@ic.gc.ca" title="E-mail the Canadian Intellectual Property Office">
                  CIPO Client Service Centre
                 </a>
                 .
                </p>
                <p>
                 Please note that files with extensions .pep and .seq that were created by CIPO as working files might be incomplete and are not to be considered official communication.
                </p>
               </div>
               <form action="/opic-cipo/cpd/eng/patent/3042857/summary.html" id="bslForm" method="post" name="bslForm">
                <input id="downloadBslFlag" name="downloadBslFlag" type="hidden" value="Y"/>
                <input id="captchaInvalidBsl" name="captchaInvalidBsl" type="hidden" value="N"/>
                <input id="bslTokenValue" name="bslTokenValue" type="hidden"/>
                <input id="bslDownloadToken" name="bslDownloadToken" type="hidden"/>
                <input id="pdrsUnavailable" name="pdrsUnavailable" type="hidden" value="N"/>
                <div class="text-center">
                 <fieldset>
                  <legend class="patents">
                   <abbr title="Biological Sequence Listings">
                    BSL Files
                   </abbr>
                  </legend>
                  <table class="table table-striped table-bordered no-footer" id="BSLTable" title="Table showing BSL files">
                   <thead>
                    <tr class="active">
                     <th scope="col">
                      File Name
                     </th>
                     <th scope="col">
                      Received On
                     </th>
                     <th scope="col">
                      Size (bytes)
                     </th>
                    </tr>
                   </thead>
                   <tbody>
                    <tr>
                     <td class="text-left">
                      <input checked="checked" id="IB201705.PEP.1" name="version" type="radio" value="3"/>
                      <label for="IB201705.PEP.1">
                       IB201705.PEP
                      </label>
                     </td>
                     <td class="text-center">
                      2019-05-03
                     </td>
                     <td class="text-right">
                      22,799
                     </td>
                    </tr>
                    <tr>
                     <td class="text-left">
                      <input id="IB201705.SEQ.2" name="version" type="radio" value="4"/>
                      <label for="IB201705.SEQ.2">
                       IB201705.SEQ
                      </label>
                     </td>
                     <td class="text-center">
                      2019-05-03
                     </td>
                     <td class="text-right">
                      9,438
                     </td>
                    </tr>
                    <tr>
                     <td class="text-left">
                      <input id="IB201705.TXT.3" name="version" type="radio" value="1"/>
                      <label for="IB201705.TXT.3">
                       IB201705.TXT
                      </label>
                     </td>
                     <td class="text-center">
                      2019-05-03
                     </td>
                     <td class="text-right">
                      89,439
                     </td>
                    </tr>
                    <tr>
                     <td class="text-left">
                      <input id="IB201705.TXT.4" name="version" type="radio" value="2"/>
                      <label for="IB201705.TXT.4">
                       IB201705.TXT
                      </label>
                     </td>
                     <td class="text-center">
                      2019-05-03
                     </td>
                     <td class="text-right">
                      89,485
                     </td>
                    </tr>
                   </tbody>
                  </table>
                 </fieldset>
                </div>
                <div class="formSectionContent" id="captchaBsl" style="margin-top: 10px;">
                 <p class="h4">
                  <strong>
                   To view selected files, please enter reCAPTCHA code :
                  </strong>
                 </p>
                 <div class="g-recaptcha" data-sitekey="6LcTxQATAAAAAHfRmJId2aDpF8XyEfAxCouBSozi" id="bslRecaptcha">
                 </div>
                 <br/>
                 <button class="btn btn-primary" type="submit">
                  Download BSL
                 </button>
                </div>
               </form>
               <script type="text/javascript">
                $(document).ready(function () {
		    $('#bslForm').submit(function () {      
		    	blockUIForBslDownload();
		    });
		  });
		  
		var bslDownloadCheckTimer;
	  	function blockUIForBslDownload() {
		    //use the current timestamp as the token value
		    var token = new Date().getTime();    
		    $('#bslTokenValue').val(token);
		    $('#downloadBslFlag').val("Y");
			document.getElementById("domMessage").setAttribute("role", "alert");	
			
			if(navigator.userAgent.indexOf("MSIE") != -1 ) {
   				var pb = document.getElementById("domMessage");
   				pb.innerHTML = '<h3>Your request is in progress.<br><br>Requested information will be available<br>in a moment.<br><br>Thank you for waiting.</h3><img src="/opic-cipo/cpd/images/progress.gif" class="image-actual text-center" alt="Request in progress image " />';    				    				    				    			
			}
			
		    $.blockUI({ 
		    	message: $('#domMessage'),
		    	css: { backgroundColor: '#FFFFFF', color: '#000000', border: '3px solid #6487C6' } 
		    	}); 
		    	
		    bslDownloadCheckTimer = window.setInterval(function () {	  
			  var cookieValue = $.cookie("bslDownloadToken");
		      if (cookieValue == token)
		       finishBslDownload();
		    }, 1000);
		  }
		  
		 function finishBslDownload() {
			 window.clearInterval(bslDownloadCheckTimer);
			 $.removeCookie('bslDownloadToken'); //clears this cookie value
			 $.unblockUI(); 
			 $('#catpchaInvalidBsl').hide();
			 grecaptcha.reset(1);
			//<!-- The following is only for Internet Explorer -->
			 if(navigator.userAgent.indexOf("MSIE") != -1 ) {
				$('#domMessage').hide();
			 }
		}
               </script>
              </div>
             </div>
            </details>
           </div>
          </form>
         </div>
        </details>
       </div>
       <script type="text/javascript">
        $(document).ready(function() {
		//alert(sessionStorage['lastclick']);
		if(sessionStorage['lastclick']===undefined){
		    console.log('undefined')
		}else{
			// delay setting the focus, it takes a while to build the tabs and if we set the focus right away the label loses the focus
			setTimeout(function(){$('#'+sessionStorage['lastclick']).focus();}, 500);
		    // once the focus is set, clear the saved id by setting it to undefined    
		    setTimeout(function(){sessionStorage['lastclick']='undefined';}, 500);
		};
	});
	
	function saveFocusId() {
		//alert(event.srcElement.id);
		sessionStorage['lastclick']=event.srcElement.id; //memorize id 
		return true;
	};
	

   $(document).ready(function () {
     $('#documentsTable').DataTable({
     "destroy": true
     ,"stateSave" : true
     ,"searching" : false
     ,"info" : false
     ,"paging": false, "lengthMenu": [[ 10, 25, 50, 100, -1], [ 10, 25, 50, 100, "All"]] ,"columnDefs": [  { "orderable": true } ], "order": [2 , "desc"]
                  
       });
   });
       </script>
       <!-- JSP pages ENDS -->
      </div>
     </div>
     <!-- Main Content ENDS -->
     <!-- SecNav STARTS -->
     <nav aria-labelledby="wb-sec-h" class="wb-sec col-lg-3 col-lg-pull-9 wb-navcurr-inited" id="wb-sec" typeof="SiteNavigationElement">
      <h2 class="wb-inv" id="wb-sec-h">
       CPD Menu
      </h2>
      <section class="list-group menu list-unstyled">
       <h3>
        Canadian Patents Database
       </h3>
       <ul class="list-group menu list-unstyled">
        <li>
         <a class="list-group-item" href="/opic-cipo/cpd/eng/introduction.html" title="Introduction">
          Introduction
         </a>
        </li>
        <li>
         <a class="list-group-item" href="/opic-cipo/cpd/eng/greenTechnologies/green_technologies.html" title="Green Technologies">
          Green Technologies
         </a>
        </li>
        <li>
         <a class="list-group-item" href="/opic-cipo/cpd/eng/searchMenu.html" title="Search">
          Search
         </a>
         <ul class="list-group menu list-unstyled">
          <li>
           <a class="list-group-item" href="/opic-cipo/cpd/eng/search/basic.html" title="Basic Search">
            Basic Search
           </a>
          </li>
          <li>
           <a class="list-group-item" href="/opic-cipo/cpd/eng/search/number.html" title="Number Search">
            Number Search
           </a>
          </li>
          <li>
           <a class="list-group-item" href="/opic-cipo/cpd/eng/search/boolean.html" title="Boolean Search">
            Boolean Search
           </a>
          </li>
          <li>
           <a class="list-group-item" href="/opic-cipo/cpd/eng/search/advanced.html" title="Advanced Search Options">
            Advanced Search
           </a>
          </li>
         </ul>
        </li>
        <li>
         <a class="list-group-item">
          <strong>
           Help
          </strong>
         </a>
         <ul class="list-group menu list-unstyled">
          <li>
           <a class="list-group-item" href="/opic-cipo/cpd/eng/help/content/help_general_content.html" target="_blank" title="General Content - opens in a new window">
            General Content
           </a>
          </li>
          <li>
           <a class="list-group-item" href="/opic-cipo/cpd/eng/help/searching/help_search_content.html" target="_blank" title="Searching - opens in a new window">
            Searching
           </a>
          </li>
          <li>
           <a class="list-group-item" href="/opic-cipo/cpd/eng/help/help_search_language.html" target="_blank" title="Search Language - opens in a new window">
            Search Language
           </a>
          </li>
          <li>
           <a class="list-group-item" href="/opic-cipo/cpd/eng/help/faq/help_faq_content.html" target="_blank" title="FAQ - opens in a new window">
            FAQ
           </a>
          </li>
         </ul>
        </li>
        <li>
         <a class="list-group-item" href="/opic-cipo/cpd/eng/disclaimer.html" target="_blank" title="Disclaimer - opens in a new window">
          Disclaimer
         </a>
        </li>
       </ul>
      </section>
     </nav>
     <!-- SecNav ENDS -->
    </div>
    <!-- SecContentBottomStart -->
    <div class="container" role="complementary">
     <div class="row pagedetails pad-bttm-sm">
      <div class="col-sm-9 col-sm-offset-3">
       <div class="col-xs-6 text-left pad-0">
        <a class="btn btn-default" href="https://cipo.ic.gc.ca/eic/site/cipointernet-internetopic.nsf/frm-eng/NGRR-8VVGD7" title="Report a problem or mistake">
         Report a problem or mistake on this page
        </a>
       </div>
      </div>
     </div>
     <div class="clearfix">
     </div>
    </div>
    <div class="container" role="complementary">
     <div class="row pagedetails pad-bttm-sm">
      <div class="col-sm-9 col-sm-offset-3">
       <div class="col-xs-6 datemod pad-0 mrgn-tp-sm">
        <!--  Dynamic pages -->
        <dl id="wb-dtmd" title="Date modified: 2021-12-09">
         <dt>
          Version number:
         </dt>
         <dd>
          3.2.14
         </dd>
        </dl>
        <!-- Static pages -->
       </div>
      </div>
     </div>
    </div>
    <div class="clearfix">
    </div>
    <!-- SecContentBottomEnd -->
    <!-- Footer STARTS -->
    <footer id="wb-info" role="contentinfo">
     <nav aria-label="Primary Footer" class="container wb-navcurr" role="navigation">
      <h2 class="wb-inv">
       About government
      </h2>
      <ul class="list-unstyled colcount-sm-2 colcount-md-3">
       <li>
        <a href="http://www.ic.gc.ca/eic/site/cipointernet-internetopic.nsf/eng/wr00006.html">
         Contact us
        </a>
       </li>
       <li>
        <a href="https://www.canada.ca/en/government/dept.html">
         Departments and agencies
        </a>
       </li>
       <li>
        <a href="https://www.canada.ca/en/government/publicservice.html">
         Public service and military
        </a>
       </li>
       <li>
        <a href="https://www.canada.ca/en/news.html">
         News
        </a>
       </li>
       <li>
        <a href="https://www.canada.ca/en/government/system/laws.html">
         Treaties, laws and regulations
        </a>
       </li>
       <li>
        <a href="https://www.canada.ca/en/transparency/reporting.html">
         Government-wide reporting
        </a>
       </li>
       <li>
        <a href="http://pm.gc.ca/eng">
         Prime Minister
        </a>
       </li>
       <li>
        <a href="https://www.canada.ca/en/government/system.html">
         How government works
        </a>
       </li>
       <li>
        <a href="http://open.canada.ca/en/">
         Open government
        </a>
       </li>
      </ul>
     </nav>
     <div class="brand">
      <div class="container">
       <div class="row">
        <nav aria-label="Secondary Footer" class="col-md-10 ftr-urlt-lnk" role="navigation">
         <h2 class="wb-inv">
          About this site
         </h2>
         <ul>
          <li>
           <a href="https://www.canada.ca/en/social.html">
            Social media
           </a>
          </li>
          <li>
           <a href="https://www.canada.ca/en/mobile.html">
            Mobile applications
           </a>
          </li>
          <li>
           <a href="http://www1.canada.ca/en/newsite.html">
            About Canada.ca
           </a>
          </li>
          <li>
           <a href="http://www.ic.gc.ca/eic/site/icgc.nsf/eng/h_07033.html">
            Terms and conditions
           </a>
          </li>
          <li>
           <a href="http://www.ic.gc.ca/eic/site/icgc.nsf/eng/h_07033.html#p1">
            Privacy
           </a>
          </li>
         </ul>
        </nav>
        <div class="col-xs-6 visible-sm visible-xs tofpg">
         <a href="#wb-cont">
          Top of Page
          <span class="glyphicon glyphicon-chevron-up">
          </span>
         </a>
        </div>
        <div class="col-xs-6 col-md-2 text-right">
         <object aria-label="Symbol of the Government of Canada" data="/app/scr/opic-cipo/boew-wet/WET4/themes-dist-gcweb/assets/wmms-blk.svg" role="img" tabindex="-1" type="image/svg+xml">
         </object>
        </div>
       </div>
      </div>
     </div>
    </footer>
    <!-- Footer ENDS -->
    <!-- Script files-->
    <script src="https://www.ic.gc.ca/app/scr/opic-cipo/boew-wet/wet/4.0.28.1/themes-dist-gcweb/js/wet-boew.min.js">
    </script>
    <script src="https://www.ic.gc.ca/app/scr/opic-cipo/boew-wet/wet/4.0.28.1/themes-dist-gcweb/js/theme.min.js">
    </script>
    <script src="https://www.ic.gc.ca/eic/home.nsf/js/jQuery.externalOpensInNewWindow.js">
    </script>
    <script type="text/javascript">
     /*<![CDATA[*/
		(function() {
			var sz = document.createElement('script');
			sz.type = 'text/javascript';
			sz.async = true;
			sz.src = 'https://siteimproveanalytics.com/js/siteanalyze_8489.js';
			var s = document.getElementsByTagName('script')[0];
			s.parentNode.insertBefore(sz, s);
		})();
    </script>
   </div>
  </div>
 </body>
</html>
